<!DOCTYPE html>
<!--  This site was created in Webflow. http://www.webflow.com  -->
<!--  Last Published: Wed Apr 22 2023 01:49:40 GMT+0000 (Coordinated Universal Time)  -->
<html data-wf-page="5e951407425f2e221853825c" data-wf-site="5e9514078c78742fa6aa5dc3">
<head>
  <meta charset="utf-8">
  <title>CompBio Workshop ICML 2023</title>
  <meta content="width=device-width, initial-scale=1" name="viewport">
  <meta content="Webflow" name="generator">
  <link href="css/normalize.css" rel="stylesheet" type="text/css">
  <link href="css/webflow.css" rel="stylesheet" type="text/css">
  <link href="css/icml-compbio.webflow.css" rel="stylesheet" type="text/css">
  <!-- [if lt IE 9]><script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.min.js" type="text/javascript"></script><![endif] -->
  <script type="text/javascript">!function(o,c){var n=c.documentElement,t=" w-mod-";n.className+=t+"js",("ontouchstart"in o||o.DocumentTouch&&c instanceof DocumentTouch)&&(n.className+=t+"touch")}(window,document);</script>
  <link href="images/favicon.png" rel="shortcut icon" type="image/x-icon">
  <link href="images/webclip.png" rel="apple-touch-icon">
  <script src="https://code.jquery.com/jquery-1.11.2.min.js"></script>
</head>
<body class="body">
  <div data-collapse="medium" data-animation="default" data-duration="300" class="navbar site-navigation w-nav">
    <div class="navbar-container w-container">
      <a href="index.html" aria-current="page" class="brand-link w-nav-brand w--current">
        <h3 id="site-title" class="site-title">ICML CompBio 2023</h3>
      </a>
      <div class="menu-button w-nav-button"></div>
    </div>
  </div>
  <div data-collapse="small" data-animation="default" data-duration="50" class="navbar-2 w-nav">
    <div class="w-container">

      <nav role="navigation" class="w-nav-menu">
	      <a href="#cfp" class="w-nav-link"> Call for submissions</a>
 	 <a href="#schedule" class="w-nav-link"> Schedule</a>
     <a href="#speaker" class="w-nav-link">Invited speakers</a>
	 <a href="#DS" class="w-nav-link">Diversity</a>
     <a href="#papers_session1" class="w-nav-link">Accepted Submissions</a>
	 <a href="#SC" class="w-nav-link">Committees</a>
     <a href="#sponsor" class="nav-link-2 w-nav-link">Sponsors</a>
	      <a href="#contact" class="w-nav-link">Contact</a>
      </nav>
      <div class="w-nav-button">
        <div class="w-icon-nav-menu"></div>
      </div>
    </div>
  </div>
  <div class="body-container w-container"></div>
  <div class="header-background">
    <div class="headline-container w-container">
      <h1 id="headline" class="heading">The 2023 ICML Workshop on Computational Biology</h1>
    </div>
  </div>
  <main id="main-content" class="header2">
    <div class="container w-container">
      <ul>
	      
<li class="list-item-4"><strong class="bold-text-2">Decision is out✨</strong></li>
<!-- <li class="list-item-3"><strong class="bold-text"><a href="https://forms.gle/Kz42r8oTroDgSJZa8" target="_blank">Apply for diversity fellowship </a></strong> <br><br></li>
 -->

	      <!-- <li class="list-item-7"><strong class="bold-text">Review deadline: June 9th, 2023 (11:59 PM Pacific Time)</strong></li>
<!-- <li class="list-item-7"><strong class="bold-text"><a href="https://cmt3.research.microsoft.com/WCB2023" target="_blank">Submission portal</a></strong></li><br> -->
	      <li class="list-item-4"><strong class="bold-text-2">Camera-ready deadline: July 21th, 2023</strong></li>
        <li class="list-item-4"><strong class="bold-text-2">Workshop time: Saturday, July 29th, 2023</strong></li> 
        <li class="list-item-3"><strong class="bold-text">Location: Hawaii Convention Center (Room 314), Honolulu, Hawai'i, USA</strong></li>
<!-- 	      <li class="list-item-3"><strong class="bold-text">Stay tuned for more information</strong></li> -->
<!-- 	<li class="list-item-7"><strong class="bold-text"> <a href="https://icml.cc/virtual/2023/workshop/13464" target="_blank">Link for virtual</a> </strong></li>     
 -->
<!--         <li class="list-item-5"><strong class="bold-text">Workshop date</strong></li> -->
<!--         <li class="list-item-7"><strong class="bold-text">Deadline:July 16th, 2023</strong></li> -->
	 <!-- <li class="list-item-7"><strong class="bold-text"> <a href="https://icml.cc/Register/view-registration" target="_blank">Registration</a> </strong></li>     -->  
   
<!-- 	      <li class="list-item-7"><strong class="bold-text">Reviewer deadline: July 16th 2023 (11:59 PM Pacific Time)</strong></li> -->
   <!-- <li class="list-item-6"><strong class="bold-text">Fellowship application (until July 20th): <a href="https://forms.gle/dfFcXHreGRAoohjx8" target="_blank">Apply</a></strong></li>
  -->        
 <!--        <li class="list-item-3"><strong class="bold-text"><a href="https://icml.cc/virtual/2023/workshop/8366" target="_blank">Click here for Workshop Livestream </a></strong> <br><br></li> -->
       </ul>
    </div>
  </main>
  <main id="about" class="header2">
    <div class="w-container">
      <div class="rich-text-block w-richtext">
   <p> Each year, machine learning (ML) advances are successfully translated to develop systems we now use regularly, such as speech recognition platforms or translation software. The COVID-19 pandemic has highlighted the urgency for translating these advances to the domain of biomedicine. Biological data has unique properties (high dimensionality, degree of noise and variability), and therefore poses new challenges and opportunities for methods development. To facilitate progress toward long-term therapeutic strategies or basic biological discovery, it is critical to bring together practitioners at the intersection of computation, ML, and biology.
The ICML Workshop on Computational Biology (WCB) will highlight how ML approaches can be tailored to making both translational and basic scientific discoveries with biological data, such as genetic sequences, cellular features or protein structures and imaging datasets, among others. This workshop thus aims to bring together interdisciplinary ML researchers working in areas such as computational genomics; neuroscience; metabolomics; proteomics; bioinformatics; cheminformatics; pathology; radiology; evolutionary biology; population genomics; phenomics; ecology, cancer biology; causality; representation learning and disentanglement to present recent advances and open questions to the machine learning community. We especially encourage interdisciplinary submissions that might not neatly fit into one of these categories. 
</p>
      </div>
    </div>
  </main>


<!-- <main id="Awards" class="main-content">
    <div class="container-2 w-container">
      <h4 class="heading-4">Awards</h4>
    </div>
    <div class="w-container">
      <p class="paragraph-2">
	<span>We are pleased to announce the following awards sponsored by: </span>
      	<h4 class="heading-6">Contributed Talk Award</h4>
	<ul>
	</ul>
      	<h4 class="heading-6">Best Spotlight Talk</h4>
	<ul>
	</ul>
	<h4 class="heading-6">Best Poster Award</h4>
	<ul>
	</ul>
    </div>
  </main>   
 -->

   <main id="Important time" class="main-content">
    <div class="container-2 w-container">
      <h4 class="heading-4">Important Dates</h4>
    </div>
    <div class="w-container">
      <p class="paragraph-2"><span>
        </span>
        Deadline for paper submissions: May <s>17th</s> 19th 2023 (11:59 PM Pacific Time)<br>
<!-- 	Deadline for edit submissions: May 16th, 2023 (11:59 PM Pacific Time)<br> -->
        Reviewer deadline: June 9th 2023 <br>
        Notification of acceptance: June 19th, 2023<br>
<!--          Video recording deadline: June 16th, 2023<br>  -->
        Camera-ready deadline: July 21th, 2023<br>
        Workshop date: July 29th, 2023 
    </div>
  </main>   
  
  
 <section id="cfp" class="header2">
    <div class="w-container">
      <h4 class="heading-2"> Call for submissions</h4>
      <p class="paragraph-6">We invite extended abstracts dealing with novel algorithms and computational approaches that are robust, 
	      scalable to high-dimensional data, and provide interpretable models of biological systems. These can be applications of ML methods or 
	      bioinformatics approaches to biological and biomedical data or novel approaches that enable new analyses. Papers will be presented in poster 
	      format and some will be selected for oral presentation (contributed and spotlight talks). 
	      Through invited talks and presentations by the participants, this workshop will bring together current advances in Computational Biology and set 
	      the stage for continuing interdisciplinary research discussions.<br>

        
<br><strong>Submission</strong>
 <br>All novel Computational Biology approaches are of interest to the workshop.<br>
	      In addition to the main track, we will hold a special track on “Explainability in Biological data” which will feature submissions focused on interpretable ML approaches and their applications for biological data. This will include inherently explainable and interpretable ML approaches, post-hoc interpretations of existing models, ways to evaluate the quality of explanations, limitations and failure modes of existing methods, and visualization strategies for analyzing models.
	     <br> We welcome original abstracts on preliminary ideas and findings in the following format:<br>
     <ul>
	<li><strong>Extended abstract not exceeding 4 pages</strong> in length (plus optional pages for references). Appendices are allowed but not guaranteed to be reviewed.</li>

     </ul>
All submissions must use the ICML WCB template with <a href="https://github.com/icml-compbio/icml-compbio.github.io/raw/master/2023/icml_wcb_2023.zip" target="_blank">this sty file</a>. 
The submission does not need to be anonymized. 
<br>Submissions should be made through the <a href="https://cmt3.research.microsoft.com/WCB2023" target="_blank">Microsoft CMT submission portal</a>. 
For more information about the submission process using CMT, please check this <a href="https://cmt3.research.microsoft.com/docs/help/author/author-submission-form.html" target="_blank">HOW-TO</a>.
<br>Accepted submissions will have the option of being published on the workshop website.For authors who do not wish their papers to be posted online or become citable, please mention this in the workshop submission.<br>
<br>This year, all presenters must attend in person.<br>
	    <!-- 
<br><strong>Instructions for revised submission</strong><br>
In your camera-ready submission please:<br>
1) Change the command in the .sty file at lines 120 -123 to <br>
\newcommand{\ICML@appearing}{The 2023 ICML Workshop on Computational Biology. Copyright 2023 by the author(s).} <br>

2) Comment \usepackage{icml2023} in the main .tex file and uncomment \usepackage[accepted]{icml2023}.<br>
Revised submissions should be made through the CMT by July 16th.

 -->
<br><br><strong>Poster</strong>
<br>This year we only do in-person poster presentation. The size for poster is recommended to be portrait orientation and up to 24"w x 36"h.<br>
 <br>


<!--   schedule  -->
<section id="schedule" class="header2">
    <div class="w-container">
        <h4 class="heading-2">Schedule</h4>
        <p class="paragraph-6">
            <!--<a href="">Click here for Workshop Livestream </a> <br><br>-->

            * Times below are in HST (GMT-10) <br>

            <style type="text/css">

.tg  {border-collapse:collapse;border-spacing:0;}
.tg td{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
  overflow:hidden;padding:10px 5px;word-break:normal;}
.tg th{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
  font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;}
.tg .tg-ycr8{background-color:#ffffff;text-align:left;vertical-align:top}
@media screen and (max-width: 767px) {.tg {width: auto !important;}.tg col {width: auto !important;}.tg-wrap {overflow-x: auto;-webkit-overflow-scrolling: touch;}}</style>

                <div class="tg-wrap"><table class="tg">
                    <tbody>
                    <tr>
                        <td style='text-align:center; vertical-align:middle', class="tg-ycr8"><span style="font-weight:400; font-style:normal;text-decoration:none;color:#000">9:00 - 9:05AM </span></td>
                        <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Opening Remarks</span></td>
                    </tr>
                    <tr>
                        <td class="tg-ycr8" colspan="4"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#434343">Session 1 (Chair: Cameron Park)</span></td>
                    </tr>
                    <tr>
                        <td style='text-align:center; vertical-align:middle', class="tg-ycr8" rowspan="2"><span style="font-weight:400; font-style:normal;text-decoration:none;color:#000">9:05 - 9:40AM</span></td>
                        <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Michael Bronstein</span></td>
                    </tr>
                    <tr>
                        <td class="tg-ycr8" colspan="3"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Geometric ML for designing new molecules</span></td>
                        <!-- <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
                    </tr>
                    <tr>
                    <tr>
                        <td style='text-align:center; vertical-align:middle', class="tg-ycr8" rowspan="2"><span style="font-weight:400; font-style:normal;text-decoration:none;color:#000">9:40 - 10:00AM</span></td>
                        <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">William Beardall</span></td>
                    </tr>
                    <tr>
                        <td class="tg-ycr8" colspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Genomic Interpreter: A Hierarchical Genomic Deep Neural Network with 1D Shifted Window Transformer</span></td>
                        <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2023/papers/WCBICML2023_paper107.pdf" target="_blank">Paper</a>]</span></td>
                        <!-- <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
                    </tr>
                    <tr>
                        <td style='text-align:center; vertical-align:middle', class="tg-ycr8" rowspan="5"><span style="font-weight:400; font-style:normal;text-decoration:none;color:#000">10:00 - 10:15AM</span></td>
                        <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Spotlight talks</span></td>
                    </tr>
                    <tr>
                        <td class="tg-ycr8" colspan="3"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Claudia Skok Gibbs - A Variational Inference Approach to Single-Cell Gene Regulatory Network Inference using Probabilistic Matrix Factorization</span></td>
<!--                        <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2023/papers/WCBICML2023_paper16.pdf" target="_blank">Paper</a>]</span></td>-->
                        <!--<td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
                    </tr>
                    <tr>
                        <td class="tg-ycr8" colspan="3"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Eric D Sun - Graph reinforcement and smoothing for improved spatial gene expression prediction</span></td>
<!--                        <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2023/papers/WCBICML2023_paper112.pdf" target="_blank">Paper</a>]</span></td>-->
<!--                          <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> &ndash;&gt;-->
                    </tr>
                    <tr>
                        <td class="tg-ycr8" colspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Jos Torge - DiffHopp: A Graph Diffusion Model for Novel Drug Design via Scaffold Hopping</span></td>
                        <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2023/papers/WCBICML2023_paper69.pdf" target="_blank">Paper</a>]</span></td>
                        <!--<td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
                    </tr>
                    <tr>
                    </tr>
                    <tr>
                        <td style='text-align:center; vertical-align:middle', class="tg-ycr8"><span style="font-weight:400; font-style:normal;text-decoration:none;color:#000">10:15 - 10:45AM </span></td>
                        <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Coffee break</span></td>
                    </tr>
                    <tr>
                        <td class="tg-ycr8" colspan="4"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#434343">Session 2 (Chair: Lingting Shi)</span></td>
                    </tr>
                    <tr>
                        <td style='text-align:center; vertical-align:middle; max-width:200px', class="tg-ycr8" rowspan="2"><span style="font-weight:400; font-style:normal;text-decoration:none;color:#000">10:45 - 11:05AM</span></td>
                        <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Rui Yang</span></td>
                    </tr>
                    <tr>
                        <td class="tg-ycr8" colspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">HiC2Self: Self-supervised Hi-C contact map denoising</span></td>
                        <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2023/papers/WCBICML2023_paper147.pdf" target="_blank">Paper</a>]</span></td>
                        <!-- <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
                    </tr>
                    <tr>
                        <td style='text-align:center; vertical-align:middle', class="tg-ycr8" rowspan="2"><span style="font-weight:400; font-style:normal;text-decoration:none;color:#000">11:05 - 11:25AM</span></td>
                        <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Sukwon Yun</span></td>
                    </tr>
                    <tr>
                        <td class="tg-ycr8" colspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Single-cell RNA-seq data imputation using Feature Propagation</span></td>
                        <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2023/papers/WCBICML2023_paper168.pdf" target="_blank">Paper</a>]</span></td>
                    </tr>
                    <tr>
                        <td style='text-align:center; vertical-align:middle', class="tg-ycr8" rowspan="6"><span style="font-weight:400; font-style:normal;text-decoration:none;color:#000">11:25 - 11:50AM</span></td>
                        <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Spotlight talks</span></td>
                    </tr>
                    <tr>
                        <td class="tg-ycr8" colspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000"></span>Kristina Ulicna - Learning dynamic image representations for self-supervised cell cycle annotation</td>
                        <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2023/papers/WCBICML2023_paper23.pdf" target="_blank">Paper</a>]</span></td>
                    </tr>
                    <tr>
                        <td class="tg-ycr8" colspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000"></span>Nhat Khang Ngo - Multiresolution Graph Transformers and Wavelet Positional Encoding for Learning Hierarchical Structures</td>
                        <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2023/papers/WCBICML2023_paper41.pdf" target="_blank">Paper</a>]</span></td>
                    </tr>
                    <tr>
                        <td class="tg-ycr8" colspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000"></span>Cuong Q Nguyen - Molecule-Morphology Contrastive Pretraining for Transferable Molecular Representation</td>
                        <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2023/papers/WCBICML2023_paper8.pdf" target="_blank">Paper</a>]</span></td>
                    </tr>
                    <tr>
                        <td class="tg-ycr8" colspan="3"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000"></span>Kavi Gupta - Improved modeling of RNA-binding protein motifs in an interpretable neural model of RNA splicing</td>
<!--                        <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2023/papers/WCBICML2023_paper120.pdf" target="_blank">Paper</a>]</span></td>-->
                        <!--<td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
                    </tr>
                    <tr>
                        <td class="tg-ycr8" colspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000"></span>Simon V Mathis - Normal Mode Diffusion: Towards Dynamics-informed Protein Design</td>
                        <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2023/papers/WCBICML2023_paper121.pdf" target="_blank">Paper</a>]</span></td>
                    </tr>
                    <tr>
                        <td style='text-align:center; vertical-align:middle', class="tg-ycr8" rowspan="1"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">11:50 - 12:50PM</span></td>
                        <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000"><a href="#papers_session1" class="w-nav-link">Poster session 1</a><!-- [<a href="https://eventhosts.gather.town/oTvLwqGTGVzInPP8/compbio-w-poster-room-1" target="_blank">Virtual World Room 1 </a>] --></span></td>
                    </tr>
                    <tr>
                        <td style='text-align:center; vertical-align:middle', class="tg-ycr8" rowspan="1"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">12:50 - 1:30PM</span></td>
                        <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Lunch break</span></td>
                    </tr>
                    <tr>
                        <td class="tg-ycr8" colspan="4"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#434343">Session 3 (Chair: Bianca M. Dumitrascu)</span></td>
                    </tr>
                    <tr>
                        <td style='text-align:center; vertical-align:middle', class="tg-ycr8" rowspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">1:30 - 2:05PM</span></td>
                        <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Bin Yu</span></td>
                    </tr>
                    <tr>
                        <td class="tg-ycr8" colspan="3"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Veridical data science for reliable, reproducible, and transparent data analysis and decision-making</span></td>
                    </tr>
                    <tr>
                        <td style='text-align:center; vertical-align:middle', class="tg-ycr8" rowspan="5"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">2:05 - 2:25PM</span></td>
                        <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Spotlight talks </span></td>
                    </tr>
                    <tr>
                        <td class="tg-ycr8" colspan="3"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000"></span>Zhongliang Zhou - Explaining the blackbox: Unraveling Protein Language Model's Learning Mechanisms for Kinase-Specific Phosphorylation Prediction</td>
                        <!--<td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
                    </tr>
                    <tr>
                        <td class="tg-ycr8" colspan="3"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Jun Xia - Why Deep Models Often Cannot Beat Non-deep Counterparts on Molecular Property Prediction?</span></td>
<!--                        <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2023/papers/WCBICML2023_paper39.pdf" target="_blank">Paper</a>]</span></td>-->
                        <!--<td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
                    </tr>
                    <tr>
                        <td class="tg-ycr8" colspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Alex M Tseng - Hierarchically branched diffusion models for scientific discovery</span></td>
                        <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2023/papers/WCBICML2023_paper102.pdf" target="_blank">Paper</a>]</span></td>
                    </tr>
                    <tr>
                        <td class="tg-ycr8" colspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Kemal Inecik - scARE: Attribution Regularization for Single Cell Representation Learning</span></td>
                        <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2023/papers/WCBICML2023_paper119.pdf" target="_blank">Paper</a>]</span></td>
                    </tr>
                    <tr>
                        <td style='text-align:center; vertical-align:middle', class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">2:25 - 3:00PM</span></td>
                        <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Invited talk: Su-In Lee</span></td>
                    </tr>
                    <tr>
                        <td style='text-align:center; vertical-align:middle', class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">3:00 - 4:00PM</span></td>
                        <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000"> <a href="#papers_session2" class="w-nav-link">Poster session 2</a> <!-- [<a href="https://eventhosts.gather.town/oTvLwqGTGVzInPP8/compbio-w-poster-room-1" target="_blank">Virtual World Room 1 </a>] --></span></td>
                    </tr>
                    <tr>
                        <td style='text-align:center; vertical-align:middle', class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">4:05 - 4:15PM</span></td>
                        <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Award ceremony and concluding remarks</span></td>
                    </tr>
                    </tbody>
                </table></div>
                </p>
            </div>


</section>



	    

    <!--   Speaker list  -->
    <header id="speaker" class="hero">
        <div class="w-container">
            <h4 class="heading-3">Invited speakers</h4>
            <p class="paragraph-10">*Listed alphabetically</p>
        </div>

        <div class="flex-container w-container"><img src="2023/img/speaker_Bronstein.jpg" width="225" alt="" class="image-3">
            <div class="hero-image-mask"></div>
            <div class="div-block-4">
                <h5 class="heading-9"><a href="https://www.cs.ox.ac.uk/people/michael.bronstein/" target="_blank">Michael Bronstein</a></h5>
                <p class="paragraph-7">
                    DeepMind Professor of Artificial Intelligence<br>
                    Department of Computer Science<br>
                    University of Oxford
                </p>
            </div>
        </div>

        <div class="flex-container w-container"><img src="2023/img/speaker_SuIn_Lee.jpg" width="225" alt="" class="image-3">
            <div class="hero-image-mask"></div>
            <div class="div-block-4">
                <h5 class="heading-9"><a href="https://aims.cs.washington.edu/su-in-lee" target="_blank">Su-In Lee</a></h5>
                <p class="paragraph-7">
                    Professor<br>
                    Paul G. Allen School of Computer Science<br>
                    University of Washington, Seattle
                </p>
            </div>
        </div>

        <div class="flex-container w-container"><img src="2023/img/speaker_BinYu.jpg" width="225" alt="" class="image-3">
            <div class="hero-image-mask"></div>
            <div class="div-block-4">
                <h5 class="heading-9"><a href="https://binyu.stat.berkeley.edu/" target="_blank">Bin Yu</a></h5>
                <p class="paragraph-7">
                    Chancellor's Distinguished Professor<br>
                    Department of Statistics & EECS<br>
                    UC Berkeley
                </p>
            </div>
        </div>

<!--    Diversity & inclusion  -->
   <main id="DS" class="main-content">
    <div class="container-2 w-container">
      <h4 class="heading-4">Diversity and Inclusion</h4>
    </div>
    <div class="w-container">
      <p class="paragraph-2"><span>
        </span>
	 
	   ICML Workshop on Computational Biology aims to foster an inclusive and welcoming community.    
	   If you have any questions, comments, or concerns, please reach out to workshopcompbio@gmail.com. <br>
     
	    
        <br>
        We are also featuring other workshops that you might find helpful for diversity and inclusion. 	      
        <a href="https://www.queerinai.com/icml" target="_blank">Queer in AI</a> 
	      
        <br>
	<br>  
<!-- 	       <strong class="bold-text">Diversity Fellowship </strong>
	<br>  
	We are pleased to announce that we will continue our Diversity Fellowship for students this year. 
        Awards include a free virtual or in-person workshop registration. We encourage applications from underrepresented groups. The deadline is July 15th. Apply <a href="https://forms.gle/HJUBooBYFn4s2sJ67" target="_blank">here</a>.   

-->
    </div>
  </main>


<!-- Accepted papers -->
<header id="papers_session1" class="hero">
    <div class="w-container">
        <h4 class="heading-3">Accepted Submissions</h4>
        <br>

        <h5 class="heading-4">Contributed talks</h5>
        <p class="paragraph-10" aligh="justify">

        <ul>
            <li>Zehui Li (Imperial College London); Akashaditya Das (Imperial College London); William Beardall (Imperial College London); Yiren Zhao (University of Cambridge); Guy-Bart Stan (Imperial College London)
                <strong>Genomic Interpreter: A Hierarchical Genomic Deep Neural Network with 1D Shifted Window Transformer
                </strong>
                [<a href="2023/papers/WCBICML2023_paper107.pdf" target="_blank">paper</a>]
            </li>
            <br>


            <li>Rui Yang (MSKCC); Alireza Karbalayghareh (MSKCC); Christina Leslie (MSKCC)
                <strong>HiC2Self: Self-supervised Hi-C contact map denoising </strong>
                [<a href="2023/papers/WCBICML2023_paper147.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Sukwon Yun (KAIST); Junseok Lee (KAIST); Chanyoung Park (KAIST)
                <strong>Single-cell RNA-seq data imputation using Feature Propagation
                </strong>
                [<a href="2023/papers/WCBICML2023_paper168.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Matthew R. Masters (University of Basel); Amr H. Mahmoud (University of Basel); Markus A. Lill (University of Basel)
                <strong>FusionDock: Physics-informed Diffusion Model for Molecular Docking
                </strong>
                [<a href="2023/papers/WCBICML2023_paper159.pdf" target="_blank">paper</a>]
            </li>
            <br>

        </ul>
        </p>

        <h5 class="heading-4">Poster session 1</h5>
        <p class="paragraph-10" align="justify">

        <ul>
            <li> Forough Habibollahi (Cortical Labs); Moein Khajehnejad (Monash University); Amitesh Gaurav (Cortical Labs); Brett J. Kagan (Cortical Labs)
                <strong> Biological Neurons vs Deep Reinforcement Learning: Sample efficiency in a simulated game-world.</strong>
                 [<a href="2023/papers/WCBICML2023_paper10.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li> Bongjin Koo (UCSB); Julien N. P. Martel (Stanford University); Ariana Peck (SLAC National Accelerator Laboratory); Axel Levy (Stanford University); Frederic Poitevin (SLAC National Accelerator Laboratory); Nina Miolane (UCSB)
                <strong> CryoChains: Heterogeneous Reconstruction of Molecular Assembly of Semi-flexible Chains from Cryo-EM Images </strong>
                 [<a href="2023/papers/WCBICML2023_paper17.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li> Pierre-Aurelien Gilliot (University of Bristol); Thomas E Gorochowski (University of Bristol)
                <strong>Come with the Sequence, Stay for the Structure: RNA Conformational Learning to Predict Protein Expression </strong>
<!--                 [<a href="2023/papers/WCBICML2023_paper19.pdf" target="_blank">paper</a>]-->
            </li>
            <br>

            <li> Yongjian Yang (Texas A&M University); Yu-Te Lin (National Taiwan University); Guanxun Li (Texas A&M University); Yan Zhong (East China Normal University); Qian Xu (Texas A&M University); James J Cai (Texas A&M University)
                <strong>Interpretable modeling of time-resolved single-cell gene-protein expression using CrossmodalNet </strong>
<!--                 Opt out [<a href="2023/papers/WCBICML2023_paper24.pdf" target="_blank">paper</a>]-->
            </li>
            <br>

            <li> Achille Nazaret (Columbia University); Justin J Hong (Columbia University); Elham Azizi (Columbia University); David Blei (Columbia University)
                <strong>Sparse Causal Discovery for Scalable Gene Regulatory Network Inference </strong>
<!--                 [<a href="2023/papers/WCBICML2023_paper25.pdf" target="_blank">paper</a>]-->
            </li>
            <br>

            <li> Daisy Yi Ding (Stanford University); Yuhui Zhang (Stanford University); Yuan Jia (Stanford University); Jiuzhi Sun (Stanford University)
                <strong>Machine Learning-guided Lipid Nanoparticle Design for mRNA Delivery </strong>
                 [<a href="2023/papers/WCBICML2023_paper36.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li> Yanis Tazi (Tri-I PhD Program: Cornell, Rockefeller, MSKCC); Michael Berger (The Rockefeller University); Winrich Freiwald (The Rockefeller University)
                <strong>Emergence of Interpretable Functional Specialization in Neural Networks Trained on Facial Expression and Identity Recognition </strong>
                 [<a href="2023/papers/WCBICML2023_paper40.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li> Frederic Runge (University of Freiburg); Jörg K.H. Franke (University of Freiburg); Frank Hutter (University of Freiburg)
                <strong>Towards Automated Design of Riboswitches </strong>
                 [<a href="2023/papers/WCBICML2023_paper42.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li> Chaitanya K Joshi (University of Cambridge); Arian R. Jamasb (University of Cambridge); Ramon Viñas Torné (University of Cambridge); Charles Harris (University of Cambridge); Simon V Mathis (University of Cambridge); Pietro Lió (University of Cambridge)
                <strong>Multi-State RNA Design with Geometric Multi-Graph Neural Networks </strong>
                 [<a href="2023/papers/WCBICML2023_paper45.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li> Kaiyu Li (University College London); Zhuo Sun (University College London)
                <strong>Multilevel Control Functional </strong>
                 [<a href="2023/papers/WCBICML2023_paper46.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li> Gurdip S Uppal (Brigham and Women's Hospital / Harvard Medical School); Georg K Gerber (Harvard Medical School)
                <strong>MC-SPACE: Microbial communities from spatially associated counts engine </strong>
                 [<a href="2023/papers/WCBICML2023_paper48.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li> Taedong Yun (Google Research)
                <strong>Evaluating unsupervised disentangled representation learning for genomic discovery and disease risk prediction </strong>
                 [<a href="2023/papers/WCBICML2023_paper54.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li> Frédéric A. Dreyer (Exscientia), Daniel Cutting  (Exscientia), Constantin Schneider (Exscientia), Henry Kenlay (Exscientia), Charlotte M. Deane (Exscientia)
                <strong>Inverse folding for antibody sequence design using deep learning </strong>
                 [<a href="2023/papers/WCBICML2023_paper61.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li> Mine Öğretir (Aalto University); Harri Lähdesmäki (Aalto University); Jamie Morton (Integrated Omics AI)
                <strong>Longitudinal Variational Autoencoder for Compositional Data Analysis </strong>
                 [<a href="2023/papers/WCBICML2023_paper67.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li> Natasa Tagasovska (Prescient Design, Genentech); Ji Won Park (Prescient Design, Genentech/Roche); Michael Maser (Prescient Design, Genentech); Stephen Ra (Prescient Design, Genentech); Kyunghyun Cho (New York University)
                <strong>BOtied: Multi-objective Bayesian optimization with tied multivariate ranks </strong>
                 [<a href="2023/papers/WCBICML2023_paper71.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li> Jing Gong (Biomap); MInsheng Hao (Tsinghua University); Xin Zeng (Biomap); Chiming Liu (Biomap); Taifeng Wang (Biomap); Jianzhu Ma (Tsinghua University); Xuegong Zhang (Tsinghua University); xingyi cheng (biomap); Le Song (Biomap & MBZUAI)
                <strong>xTrimoGene: An Efficient and Scalable Representation Learner for Single-Cell RNA-Seq Data </strong>
                 [<a href="2023/papers/WCBICML2023_paper73.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li> Alexandru Dumitrescu (Aalto University); Emmi Jokinen (University of Helsinki); Dani Korpela (Aalto University); Harri Lähdesmäki (Aalto University)
                <strong>Structure-guided T cell receptor and epitope interaction prediction </strong>
                 [<a href="2023/papers/WCBICML2023_paper75.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li> Gal Keinan (GSK); Karen Sayal (GSK.ai); Alon Gonen (GSK); Jiang Zhu (GSK); Lena Granovsky (GSK); Jeremy L England (GSK)
                <strong>Learning Perturbation-specific Cell Representations for Prediction of Transcriptional Response across Cellular Contexts </strong>
                 [<a href="2023/papers/WCBICML2023_paper85.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Divya Koyyalagunta (MSKCC); Quaid Morris (MSKCC), Vector Institute, University of Toronto)
                <strong>Gradient-based Migration History Inference of Metastatic Cancers </strong>
<!--                 Opt-out [<a href="2023/papers/WCBICML2023_paper87.pdf" target="_blank">paper</a>]-->
            </li>
            <br>

            <li>Mingxuan Zhang (Columbia University); Kevin Hoffer-Hawlik (Columbia University); Benjamin Izar (Columbia University Medical Center); Elham Azizi (Columbia University)
                <strong>SBGGeHPF: Stochastic Block Graph Generative Hierarchical Poisson Factorization </strong>
<!--                 [<a href="2023/papers/WCBICML2023_paper94.pdf" target="_blank">paper</a>]-->
            </li>
            <br>

            <li>Antonia Boca (University of Cambridge); Simon V Mathis (University of Cambridge)
                <strong>Predicting protein variants with equivariant graph neural networks </strong>
                 [<a href="2023/papers/WCBICML2023_paper96.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Yunha Hwang (Harvard); Andre Cornman (N/A); Elizabeth Kellogg (Cornell); Sergey Ovchinnikov (Harvard); Peter Girguis (Harvard)
                <strong>Genomic language model predicts protein co-regulation and function </strong>
<!--                 [<a href="2023/papers/WCBICML2023_paper105.pdf" target="_blank">paper</a>]-->
            </li>
            <br>

            <li>Ioana Lia (Columbia University); Mathini Vaikunthan (Columbia University); Andrew Redenti (Columbia University); Jongwon Im (Columbia University); Tal Danino (Columbia University); José L. McFaline-Figueroa (Columbia University); Elham Azizi (Columbia University)
                <strong>BacTIME: Computational inference of bacterial interactions with the tumor microenvironment </strong>
                 [<a href="2023/papers/WCBICML2023_paper115.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Ron Boger (UC Berkeley); Seyone Chithrananda (UC Berkeley); Amy X Lu (UC Berkeley)
                <strong>TOPH (True Retrieval of Proteins Homologs): Adapting A Contrastive Question-Answering Framework for Protein Search </strong>
                 [<a href="2023/papers/WCBICML2023_paper138.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Dennis Tang (Duke University); Jian Zhou (UT Southwestern)
                <strong>Accelerating Systematic Prediction of Variant Effects and Sequence Interpretation with Multiplexer Models </strong>
                 [<a href="2023/papers/WCBICML2023_paper139.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Michael Brocidiacono (UNC); David Koes (University of Pittsburgh); Alexander Tropsha (UNC); Konstantin Popov (UNC)
                <strong>PLANTAIN: Using an AI pose scoring function for fast and accurate molecular docking </strong>
                 [<a href="2023/papers/WCBICML2023_paper140.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Claudia Skok Gibbs (New York University); Omar Mahmood (New York University); Richard Bonneau (New York University); Kyunghyun Cho (New York University)
                <strong>A Variational Inference Approach to Single-Cell Gene Regulatory Network Inference using Probabilistic Matrix Factorization </strong>
<!--  Opt-out               [<a href="2023/papers/WCBICML2023_paper16.pdf" target="_blank">paper</a>]-->
            </li>
            <br>

            <li>Eric D Sun (Stanford University); Rong Ma (Stanford University); James Zou (Stanford University)
                <strong>Graph reinforcement and smoothing for improved spatial gene expression prediction </strong>
<!--                 [<a href="2023/papers/WCBICML2023_paper112.pdf" target="_blank">paper</a>]-->
            </li>
            <br>

            <li> Jos Torge (University of Cambridge); Charles Harris (University of Cambridge); Simon V Mathis (University of Cambridge); Pietro Lió (University of Cambridge)
                <strong>DiffHopp: A Graph Diffusion Model for Novel Drug Design via Scaffold Hopping </strong>
                 [<a href="2023/papers/WCBICML2023_paper69.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Rui Yang (MSKCC); Alireza Karbalayghareh (MSKCC); Christina Leslie (MSKCC)
                <strong>HiC2Self: Self-supervised Hi-C contact map denoising </strong>
                 [<a href="2023/papers/WCBICML2023_paper147.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Tony Tu (Georgia Tech); Gautham Krishna Sankar Ramalaxmi (Georgia Tech); Amirali Aghazadeh (Georgia Tech)
                <strong>ProtiGeno: a prokaryotic short gene finder using protein language models </strong>
                 [<a href="2023/papers/WCBICML2023_paper161.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Colin Grambow (Genentech Research and Early Development); Hayley Weir (Genentech Research and Early Development); Nathaniel Diamant (Genentech Research and Early Development); Alex M Tseng (Genentech); Tommaso Biancalani (Genentech Research and Early Development); Gabriele Scalia (Genentech); Kangway V Chuang (Genentech Research and Early Development)
                <strong>RINGER: Rapid Conformer Generation for Macrocycles with Sequence-Conditioned Internal Coordinate Diffusion </strong>
<!--  Opt-out               [<a href="2023/papers/WCBICML2023_paper146.pdf" target="_blank">paper</a>]-->
            </li>
            <br>

            <li> Xuezhi Xie (University of Toronto); Jin Sub Lee (University of Toronto); Dongki Kim (KAIST); Jaehyeong Jo (KAIST); Jisun Kim (University of Toronto); Philip Kim (University of Toronto)
                <strong>Antibody-SGM: Antigen-Specific Joint Design of Antibody Sequence and Structure using Diffusion Models </strong>
                 [<a href="2023/papers/WCBICML2023_paper143.pdf" target="_blank">paper</a>]
            </li>
            <br>

        </ul>
    </div>
</header>

<header id="papers_session2" class="hero">
    <div class="w-container">
        <h5 class="heading-4">Poster session 2</h5>
        <p class="paragraph-10" align="justify">

        <ul>
            <li>Jörg K.H. Franke (University of Freiburg); Frederic Runge (University of Freiburg); Frank Hutter (University of Freiburg)
                <strong>Scalable Deep Learning for RNA Secondary Structure Prediction </strong>
                 [<a href="2023/papers/WCBICML2023_paper43.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Talal Widatalla (Merck & Co Inc); Zachary A Rollins (Merck & Co Inc); Alan Cheng (Merck & Co Inc); Andrew Waight (Merck & Co Inc); Ming-Tang Chen (Merck & Co Inc)
                <strong>AbPROP: Language and Graph Deep Learning for Antibody Property Prediction </strong>
                 [<a href="2023/papers/WCBICML2023_paper53.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Georg K Gerber (Harvard Medical School); Michael Glickman (Weill Cornell Medicine); Vanni Bucci (U Mass Medical School)
                <strong>Discovery of Host-Microbiome Interactions Using Multi-Modal, Sparse, Time-Aware, Bayesian Network-Structured Neural Topic Models </strong>
                 [<a href="2023/papers/WCBICML2023_paper57.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Joshua Southern (Imperial College London); Arne Schneuing (EPFL); Michael Bronstein (University of Oxford); Bruno Correia (EPFL)
                <strong>Evaluation Metrics for Protein Structure Generation </strong>
                 [<a href="2023/papers/WCBICML2023_paper70.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Lovro Vrček (Genome Institute of Singapore, A*STAR); Xavier Bresson (National University of Singapore); Thomas Laurent (Loyola Marymount University); Martin Schmitz (A*Star/NUS); Mile Sikic (Genome Institute of Singapore)
                <strong>Untangling Genome Assembly Graphs with Graph Neural Networks </strong>
<!--                 Opt out[<a href="2023/papers/WCBICML2023_paper72.pdf" target="_blank">paper</a>] -->
            </li>
            <br>

            <li> Ramon Viñas Torné (University of Cambridge); Paul M Scherer (University of Cambridge); Nikola Simidjievski (University of Cambridge ); Mateja Jamnik (University of Cambridge); Pietro Lió (University of Cambridge)
                <strong>Spatio-relational inductive biases in spatial cell-type deconvolution</strong>
                 [<a href="2023/papers/WCBICML2023_paper88.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Lingting Shi (Columbia University); Michael Pressler (Columbia University); Xumin Shen (Columbia University); Joshua S Fuller (Columbia University ); Achille O R Nazaret (Columbia University); Joy Fan (Columbia University); Ajna Uzuni (Columbia University); Ximi Wang (Columbia University); David Harle (Columbia University); Ran Reshef (Columbia University); Elham Azizi (Columbia University)
                <strong>Characterizing the temporal evolution of alloreactive T cell clonotypes using probabilistic basis decomposition </strong>
<!--                 [<a href="2023/papers/WCBICML2023_paper132.pdf" target="_blank">paper</a>]-->
            </li>
            <br>

            <li>Dobrik G. Georgiev (University of Cambridge); Ramon Viñas Torné (University of Cambridge); Sam Considine (University of Cambridge); Bianca M Dumitrascu (Princeton University); Pietro Lió (University of Cambridge)
                <strong>NARTI: Neural Algorithmic Reasoning for Trajectory Inference </strong>
                 [<a href="2023/papers/WCBICML2023_paper77.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Andre Lamurias (NOVA LINCS, NOVA School of Science and Technology); Alessandro Tibo (Aalborg University); Katja Hose (TU Wien); Mads Albertsen (Aalborg University); Thomas Dyhre Nielsen (Aalborg University)
                <strong>Graph Neural Networks for Metagenomic Binning </strong>
                 [<a href="2023/papers/WCBICML2023_paper81.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Ziqi Tang (Cold Spring Harbor Laboratory); Peter K. Koo (Cold Spring Harbor Laboratory)
                <strong>Building foundation models for regulatory genomics requires rethinking large language models </strong>
                 [<a href="2023/papers/WCBICML2023_paper92.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Philip Fradkin (Vector Institute); Leo J Lee (University of Toronto); Bo Wang (Vector Institute); Brendan Frey (U. Toronto)
                <strong>Splicing Up Your Predictions with RNA Contrastive Learning </strong>
<!--                 [<a href="2023/papers/WCBICML2023_paper93.pdf" target="_blank">paper</a>]-->
            </li>
            <br>

            <li>Justin S Diamond (University of Basel)
                <strong>Geometric Constraints from Molecular Dynamics to Graph Generative Modelling </strong>
                 [<a href="2023/papers/WCBICML2023_paper97.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Dylan Domenico (MSKCC); Georgios Asimomitis (MSKCC, National Technical University of Athens); Gunes Gundem (MSKCC); Emily Stockfisch (MSKCC); Cassidy Cobbs (MSKCC); Neeman Mohibullah (MSKCC); Leonidas Alexopoulos (National Technical University of Athens); Neerav Shukla (MSKCC); Andrew Kung (MSKCC); Elli Papaemmanuil (MSKCC)
                <strong>High-Resolution Interpretable Classification of Artifacts versus Real Variants in Whole Genome Sequencing Data from Archived Tissue </strong>
                 [<a href="2023/papers/WCBICML2023_paper116.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Daniel McNeela (University of Wisconsin, Madison); Frederic Sala (University of Wisconsin-Madison); Anthony Gitter (University of Wisconsin, Madison)
                <strong>Mixed Curvature Representation Learning for Biological Pathway Graphs </strong>
                 [<a href="2023/papers/WCBICML2023_paper117.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Micaela E Consens (University of Toronto); Alan Moses (University of Toronto); Bo Wang (Vector Institute); Nicholas Papernot (University of Toronto)
                <strong>Transforming Genomic Interpretability: A DNABERT Case Study </strong>
                 [<a href="2023/papers/WCBICML2023_paper118.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Delaram Pir Hayatifard (Rice University); Elizabeth Hanson (Baylor College of Medicine ); Paul Pfaffinger (Baylor College of Medicine); Benjamin Arenkiel (Baylor College of Medicine); Jacob Reimer (Baylor College of Medicine); Saket Navlakha (Cold Spring Harbor Laboratory); Anki Patel (Rice University)
                <strong>Dense Odor Coding in the Mouse Olfactory Bulb </strong>
                 [<a href="2023/papers/WCBICML2023_paper122.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Martin Rohbeck (DKFZ); Arber Qoku (DKFZ); Tim Treis (DKFZ); Fabian J Theis (Institute of Computational Biology, Helmholtz Center); Britta Velten (Heidelberg University); Florian Buettner (DKFZ); Oliver Stegle (DKFZ)
                <strong>Cellij: A Modular Factor Model Framework for Interpretable and Accelerated Multi-Omics Data Integration </strong>
                 [<a href="2023/papers/WCBICML2023_paper124.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Osama Abdin (University of Toronto); Philip Kim (University of Toronto)
                <strong>Hypernetwork-conditioned diffusion enables all-atom peptide conformational sampling </strong>
<!--                 Opt out [<a href="2023/papers/WCBICML2023_paper125.pdf" target="_blank">paper</a>] -->
            </li>
            <br>

            <li>Sepideh Maleki (Genentech Research and Early Development); Ehsan Hajiramezalani (Genentech Research and Early Development); Gabriele Scalia (Genentech); Tommaso Biancalani (Genentech Research and Early Development); Kangway V Chuang (Genentech Research and Early Development)
                <strong>Learning to Explain Hypergraph Neural Networks </strong>
<!--                 [<a href="2023/papers/WCBICML2023_paper126.pdf" target="_blank">paper</a>]-->
            </li>
            <br>

            <li>Junseok Lee (KAIST); Sungwon Kim (KAIST); Dongmin Hyun (POSTECH); Namkyeong Lee (Korea Advanced Institute of Science and Technology); Yejin Kim (University of Texas Health Science Center at Houston); Chanyoung Park (KAIST)
                <strong>Deep single-cell RNA-seq data clustering with graph prototypical contrastive learning </strong>
                 [<a href="2023/papers/WCBICML2023_paper129.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Romain Lacombe (Stanford University); Andrew Gaut (Stanford University)
                <strong>Extracting Molecular Properties from Natural Language with Multimodal Contrastive Learning </strong>
                 [<a href="2023/papers/WCBICML2023_paper131.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Wenliang Zhong (University of Texas at Arlington); Hehuan Ma (University of Texas at Arlington); Yu Rong (Tencent AI Lab); Yatao Bian (Tencent AI Lab); Long-Kai Huang (Tencent AI Lab); Yuzhi Guo (University of Texas at Arlington); Peilin Zhao (Tencent AI Lab); Junzhou Huang (University of Texas at Arlington)
                <strong>CoSSL: A Context-based Semi-Supervised Framework for Molecular Property Prediction </strong>
<!--                 Opt-out [<a href="2023/papers/WCBICML2023_paper149.pdf" target="_blank">paper</a>] -->
            </li>
            <br>

            <li>Core Francisco Park (Harvard University); Pavel Potocek (Thermo Fisher Scientific); Shashata Sawmya (MIT); Yicong Li (Harvard University); Richard Schalek (Harvard University); Jeff Lichtman (Harvard University); Maurice Peemen (Thermo Fisher Scientific); Aravinthan Samuel (Harvard); Nir Shavit (Massachusetts Institute of Technology); Yaron Meirovitch (Harvard; MIT)
                <strong>SmartEM: Faster Connectomics Using Deep Error Prediction Based Rescanning </strong>
<!--                 Opt-out [<a href="2023/papers/WCBICML2023_pape152.pdf" target="_blank">paper</a>] -->
            </li>
            <br>

            <li>Joowon Lee (University of Wisconsin-Madison); Hanbaek Lyu (UW-Madison); Weixin Yao (University of California-Riverside)
                <strong>Interpretable Feature Extraction by Supervised Dictionary Learning for Identification of Cancer-Associated Gene Clusters </strong>
                 [<a href="2023/papers/WCBICML2023_paper155.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Kristina Ulicna (The Alan Turing Institute); Manasi Kelkar (University College London (UCL)); Christopher J Soelistyo (The Alan Turing Institute); Guillaume Charras (University College London (UCL)); Alan R Lowe (University College London)
                <strong>Learning dynamic image representations for self-supervised cell cycle annotation </strong>
                 [<a href="2023/papers/WCBICML2023_paper23.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Khang Nhat Ngo (FPT Software AI Center); Truong Son Hy (University of Chicago); Risi Kondor (The University of Chicago)
                <strong>Multiresolution Graph Transformers and Wavelet Positional Encoding for Learning Hierarchical Structures </strong>
                 [<a href="2023/papers/WCBICML2023_paper41.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Cuong Q Nguyen (GSK); Dante Pertusi (GSK); Kim M Branson (GSK)
                <strong>Molecule-Morphology Contrastive Pretraining for Transferable Molecular Representation </strong>
                 [<a href="2023/papers/WCBICML2023_paper8.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Kavi Gupta (MIT); Chenxi Yang (UT Austin); Kayla McCue (MIT); Osbert Bastani (University of Pennsylvania); Phillip Sharp (MIT); Christopher Burge (MIT); Armando Solar-Lezama (MIT)
                <strong>Improved modeling of RNA-binding protein motifs in an interpretable neural model of RNA splicing </strong>
<!--                 Opt-out [<a href="2023/papers/WCBICML2023_paper120.pdf" target="_blank">paper</a>] -->
            </li>
            <br>

            <li>Urszula Julia Komorowska (University of Cambridge); Simon V Mathis (University of Cambridge); Mateja Jamnik (University of Cambridge); Pietro Lió (University of Cambridge)
                <strong>Normal Mode Diffusion: Towards Dynamics-informed Protein Design </strong>
                 [<a href="2023/papers/WCBICML2023_paper121.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Zhongliang Zhou (University of Georgia); Wayland Yeung (University of Georgia); Saber Soleymani (University of Georgia); Nathan Gravel (University of Georgia); Mariah Salcedo (University of Georgia); Sheng Li (University of Virginia); Natarajan Kannan (University of Georgia)
                <strong>Explaining the blackbox - Unraveling Protein Language Model's Learning Mechanisms for Kinase-Specific Phosphorylation Prediction </strong>
                <!-- Opt-out [<a href="2023/papers/WCBICML2023_paper.pdf" target="_blank">paper</a>] -->
            </li>
            <br>

            <li>Jun Xia (Westlake University); Lecheng Zhang (Westlake University); Xiao Zhu (Beijing University of Posts and Telecommunications); Stan Z. Li (Westlake University)
                <strong>Why Deep Models Often Cannot Beat Non-deep Counterparts on Molecular Property Prediction? </strong>
<!--                 [<a href="2023/papers/WCBICML2023_paper39.pdf" target="_blank">paper</a>]-->
            </li>
            <br>

            <li>Alex M Tseng (Genentech); Max Shen (Genentech); Tommaso Biancalani (Genentech); Gabriele Scalia (Genentech)
                <strong>Hierarchically branched diffusion models for scientific discovery </strong>
                 [<a href="2023/papers/WCBICML2023_paper102.pdf" target="_blank">paper</a>]
            </li>
            <br>

            <li>Kemal Inecik (Institute of Computational Biology, Helmholtz Center); Fabian J Theis (Institute of Computational Biology, Helmholtz Center)
                <strong>scARE: Attribution Regularization for Single Cell Representation Learning </strong>
                 [<a href="2023/papers/WCBICML2023_paper119.pdf" target="_blank">paper</a>]
            </li>
        </ul>
    </div>
</header>


<main id="SC" class="main-content">
    <div class="container-2 w-container">
      <h4 class="heading-4">Steering committee</h4>
    </div>
    <div class="w-container">
      <p class="paragraph-2"><span>
        </span>
Dana Pe’er (mskcc.org/research/ski/labs/dana-pe-er)<br>
Debora Marks (https://www.deboramarkslab.com/)<br>
Alexander Anderson  (labpages.moffitt.org/andersona/)<br>
Elham Azizi (https://www.azizilab.com/ )<br>
Sandhya Prabhakaran  (sandhyaprabhakaran.com)<br>
Abdoulaye Baniré Diallo (labo.bioinfo.uqam.ca )<br>
Wesley Tansey  (http://wesleytansey.com)  <br>
Bianca Dumitrascu (https://computational-morphogenomics-group.github.io)<br>
Maria Brbic (https://brbiclab.epfl.ch/)
	</p>
    </div>
  </main>   
  
  <main id="OC" class="main-content">
    <div class="container-2 w-container">
      <h4 class="heading-4">Organizing committee</h4>
    </div>
    <div class="w-container">
      <p class="paragraph-2"><span>
        </span>
Yubin Xie: yux2009@med.cornell.edu <br>
Cassandra Burdziak: burdziac@mskcc.org <br>
Mafalda Dias: mafalda.dias@crg.eu<br>
Cameron Park: cyp2111@columbia.edu<br>
Pascal Notin: pascal.notin@cs.ox.ac.uk <br>
Joy Fan:	lf2684@columbia.edu <br>
Ruben Weizman:	rubenweitzman@gmail.com <br>
Lingting Shi: ls3456@columbia.edu <br>
Siyu He: sh3846@columbia.edu <br>
Yinuo Jin: yj2589@columbia.edu <br>
	    </p> 
    </div>
  </main>


<main id="main-content" class="main-content-2">
    <div class="w-container">
      <div class="rich-text-block-4 w-richtext">
        <h4>Program Committee</h4>
        <p class="paragraph-3">
            Achille Nazaret, Columbia University<br>
            Adam Gayoso, UC Berkeley<br>
            Adrian Heilbut, logphase research<br>
            Ahmed Halioui, Mt Intelligent Machines<br>
            Ahmet Coskun, Georgia Tech<br>
            Alice Bizeul, ETHZ<br>
            Alyssa Morrow, UC Berkeley<br>
            Amine Remita, Université du Québec à Montréal<br>
            Amy Xie, Memorial Sloan Kettering Cancer Center<br>
            Anastasiya Belyaeva, Google Research<br>
            Arnav Das, University of Washington<br>
            Ayshwarya Subramanian, Broad Institute<br>
            Behrooz Tahmasebi, MIT<br>
            Benjamin Gallusser, EPFL<br>
            Benjamin Wesley, Columbia University<br>
            Bianca Dumitrascu, Columbia University<br>
            Bishnu Sarker, Inria<br>
            Bo Yuan, Harvard University<br>
            Brian Trippe, Columbia University<br>
            Cameron Park, Columbia University<br>
            Cassandra Burdziak, Memorial Sloan Kettering Cancer Center<br>
            Chandana Rajesh, Cold Spring Harbor Laboratory<br>
            Charles Harris, University of Cambridge<br>
            Chenlian Fu, Memorial Sloan Kettering Cancer Center<br>
            Chetanya Pandya, 2seventy bio<br>
            Christopher Garay, Paradigm4<br>
            Dana Pe'er, Memorial Sloan Kettering Cancer Center<br>
            Debora Marks, Harvard University<br>
            Delasa Aghamirzaie, Natera<br>
            Ece Ozkan, MIT<br>
            Ehsan Hajiramezanali, Genentech<br>
            Elham Azizi, Columbia University<br>
            Elior Rahmani, UCLA<br>
            Engelbert Mephu Nguifo, Université  Clermont Auvergne, CNRS, LIMOS<br>
            Evan Seitz, Cold Spring Harbor Laboratory<br>
            Eyal Itskovits, GSK.ai<br>
            Fabian Theis, Helmholtz Zentrum München<br>
            Filippo Utro, IBM THOMAS J. WATSON RESEARCH CENTER<br>
            Freddie Bickford Smith, University of Oxford<br>
            Gabriele Scalia, Genentech<br>
            Han Spinner, Harvard<br>
            Hayda Almeida, UQAM<br>
            Ignacio Vazquez-Garcia, Memorial Sloan Kettering Cancer Center / Columbia University<br>
            Ilyes Baali, Memorial Sloan Kettering Cancer Center<br>
            Jan Brauner, University of Oxford<br>
            Jason Dou, University of Pittsburgh<br>
            Jessica White, Memorial Sloan Kettering Cancer Center<br>
            Jill Gallaher, Moffitt Cancer Center<br>
            Joy Fan, Columbia University<br>
            Justin Hong, Columbia University<br>
            Kangway Chuang, Genentech Research and Early Development<br>
            Kevin Hoffer-Hawlik, Columbia University<br>
            Keyur Shah, Massachusetts General Hospital<br>
            Khanh Dinh, Columbia University<br>
            Kyle Swanson, Stanford University<br>
            Lea Goetz, GSK.ai<br>
            Lei Xiong, MIT<br>
            Lingting Shi, Columbia University<br>
            Lingyi Cai, Columbia University<br>
            Lood van Niekerk, Harvard Medical School<br>
            Luis Rueda, University of Windsor<br>
            Manu Setty, SKI<br>
            Maria Brbic, Stanford University<br>
            Mario Wieser, Genedata AG<br>
            Matthew Peterson, Paradigm4<br>
            Mervin Fansler, Memorial Sloan Kettering Cancer Center<br>
            Mingxuan Zhang, Columbia University<br>
            Mohammad Lotfollahi, Helmholtz Zentrum München<br>
            Mostafa Karimi, Amazon<br>
            Namkyeong Lee, Korea Advanced Institute of Science and Technology<br>
            Nathan Rollins, Seismic Therapeutic<br>
            Nazim Bouatta, Harvard University<br>
            Nicolas Beltran, Columbia University<br>
            Petko Fiziev, Illumina, Inc<br>
            Philip Fradkin, Vector Institute<br>
            Pierre Boyeau, UC Berkeley<br>
            Pooja Kathail, UC Berkeley<br>
            Ragothaman Yennamalli, SASTRA Deemed University<br>
            Ričards Marcinkevičs, ETH Zurich<br>
            Rishabh Anand National, University of Singapore<br>
            Romain Lopez, Genentech and Stanford<br>
            Rui Yang, Memorial Sloan Kettering Cancer Center<br>
            Russell Kunes, Columbia University<br>
            Sabeur Aridhi, LORIA<br>
            Sairam Behera, Baylor College of Medicine<br>
            Sandhya Prabhakaran, Moffitt Cancer Center<br>
            Sepideh Maleki, Genentech<br>
            Sheng Liu, Indiana University School of Medicine<br>
            Shiyi Yang, UC Berkeley<br>
            Shouvik Mani, Columbia University<br>
            Shreshth Malik, University of Oxford<br>
            Shuhao Zhang, EPFL<br>
            Simon Mathis, University of Cambridge<br>
            Siyu He, Columbia University<br>
            Smita Krishnaswamy, Yale University<br>
            Somesh Mohapatra, Caterpillar Inc<br>
            Soufiane Mourragui, Hubrecht Institute<br>
            Soumya Kundu, Stanford University<br>
            Srivamshi Pittala, Katana Graph Inc<br>
            Stephen Zhang, University of Melbourne<br>
            Surag Nair, Stanford University<br>
            Talip Ucar, AstraZeneca<br>
            Tianming Zhou, Carnegie Mellon University<br>
            Umberto Lupo, EPFL<br>
            Victor Greiff, University of Oslo<br>
            Warith Eddine,  DJEDDI FST Manar<br>
            Xiang Niu, Memorial Sloan Kettering Cancer Center<br>
            Xue Long Zhao, University of Pennsylvania<br>
            Xuecong Fu, Carnegie Mellon University<br>
            Yanay Rosen, Stanford University<br>
            Yinuo Jin, Columbia University<br>
            Yongju Lee, Genentech<br>
            Yubin Xie, Memorial Sloan Kettering Cancer Center<br>
            Yue Wu, Stanford University<br>
            Ziqi Tang, Cold Spring Harbor Laboratory<br>
        </p>
      </div>
    </div>
  </main>

        <section id="sponsor" class="cards-section">
            <div class="w-container">
                <h4 class="heading-4">Sponsors</h4>
            </div>
            <div class="centered-container w-container">

                <div class="cards-grid-container">


                    <div id="w-node-0d05fd79f6e4-1853825c" class="div-block-2">
                        <div class="cards-image-mask"><img src="images/2021/CZ_LOGO.png" alt="" class="cards-image"></div>
                        <h6 class="heading-7"><a href="https://chanzuckerberg.com" target="_blank">Chan Zuckerberg Initiative</a></h6>
                    </div>

                    <div id="w-node-0d05fd79f6eb-1853825c" class="div-block">
                        <div class="cards-image-mask"><img src="2022/img/GSK_logo.jpeg" alt="psf" class="cards-image" style="width:256px;height:84px;"></div>
                        <h6 class="heading-6"><a href="https://www.gsk.com/en-gb/home/" class="link-3" target="_blank">GSK</a></h6>
                    </div>

                </div>
            </div>
        </section>




        <main id="main-content" class="main-content-2">
    <div class="w-container">
      <div class="rich-text-block-3 w-richtext">
        <h4>Previous workshops</h4>
        <ul>
	<li><a href="https://icml-compbio.github.io/icml-website-2022/" target="_blank">ICML 2022 Workshop on Computational Biology</a></li>
          <li><a href="https://icml-compbio.github.io/icml-website-2021/" target="_blank">ICML 2021 Workshop on Computational Biology</a></li>
          <li><a href="https://icml-compbio.github.io/icml-website-2020/" target="_blank">ICML 2020 Workshop on Computational Biology</a></li>
          <li><a href="https://sites.google.com/view/icml-compbio-2019/" target="_blank">ICML 2019 Workshop on Computational Biology</a></li>
          <li><a href="https://sites.google.com/view/wcb2018/" target="_blank">Joint ICML and IJCAI 2018 Workshop on Computational Biology</a></li>
          <li><a href="https://www.google.com/url?q=https%3A%2F%2Fsites.google.com%2Fview%2Fcompbioworkshopicml2017%2Fhome&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNFWzRemcmyQ-NMnfr5nzoYSaBOgcA" target="_blank">ICML 2017 Workshop on Computational Biology</a></li>
          <li><a href="https://www.google.com/url?q=https%3A%2F%2Fsites.google.com%2Fsite%2Fcompbioworkshopicml2016%2F&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNFjnboz8_oyLofUYTQv1Aq_MxW0nA" target="_blank">ICML 2016 Workshop on Computational Biology</a></li>
        </ul>
      </div>
    </div>
  </main>
  <footer id="contact" class="footer">
    <div class="container-3 w-container">
      <h4 class="heading-3">Contact<a href="mailto:workshopcompbio@gmail.com"></a></h4>
    </div>
    <div class="w-container">
      <p class="paragraph-3">For workshop-related queries please contact: <a href="mailto:workshopcompbio@gmail.com?subject=Question%20about%20ICML%202320%20Computational%20Biology%20workshop%20">workshopcompbio@gmail.com</a></p>
    </div>
    <div class="w-container">
      <div class="text-block">Copyright © 2023 ICML compbio workshop. All rights reserved.<br>Designed by Yubin Xie</div>
    </div>
  </footer>
<!--   <script src="https://d3e54v103j8qbb.cloudfront.net/js/jquery-3.4.1.min.220afd743d.js?site=5e9514078c78742fa6aa5dc3" type="text/javascript" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script> -->
  <script src="js/webflow.js" type="text/javascript"></script>
  <!-- [if lte IE 9]><script src="https://cdnjs.cloudflare.com/ajax/libs/placeholders/3.0.2/placeholders.min.js"></script><![endif] -->
</body>
</html>
 <br>
