<!DOCTYPE html>
<!--  This site was created in Webflow. http://www.webflow.com  -->
<!--  Last Published: Wed Apr 22 2022 01:49:40 GMT+0000 (Coordinated Universal Time)  -->
<html data-wf-page="5e951407425f2e221853825c" data-wf-site="5e9514078c78742fa6aa5dc3">
<head>
  <meta charset="utf-8">
  <title>CompBio Workshop ICML 2022</title>
  <meta content="width=device-width, initial-scale=1" name="viewport">
  <meta content="Webflow" name="generator">
  <link href="css/normalize.css" rel="stylesheet" type="text/css">
  <link href="css/webflow.css" rel="stylesheet" type="text/css">
  <link href="css/icml-compbio.webflow.css" rel="stylesheet" type="text/css">
  <!-- [if lt IE 9]><script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.min.js" type="text/javascript"></script><![endif] -->
  <script type="text/javascript">!function(o,c){var n=c.documentElement,t=" w-mod-";n.className+=t+"js",("ontouchstart"in o||o.DocumentTouch&&c instanceof DocumentTouch)&&(n.className+=t+"touch")}(window,document);</script>
  <link href="images/favicon.png" rel="shortcut icon" type="image/x-icon">
  <link href="images/webclip.png" rel="apple-touch-icon">
  <script src="https://code.jquery.com/jquery-1.11.2.min.js"></script>
</head>
<body class="body">
  <div data-collapse="medium" data-animation="default" data-duration="300" class="navbar site-navigation w-nav">
    <div class="navbar-container w-container">
      <a href="index.html" aria-current="page" class="brand-link w-nav-brand w--current">
        <h3 id="site-title" class="site-title">ICML CompBio 2022</h3>
      </a>
      <div class="menu-button w-nav-button"></div>
    </div>
  </div>
  <div data-collapse="small" data-animation="default" data-duration="50" class="navbar-2 w-nav">
    <div class="w-container">

      <nav role="navigation" class="w-nav-menu">
	      <a href="#cfp" class="w-nav-link"> Call for submissions</a>
	 <a href="#schedule" class="w-nav-link"> Schedule</a>
<a href="#speaker" class="w-nav-link">Invited speakers</a>  
	 <a href="#DS" class="w-nav-link">Diversity</a>
 <a href="#papers" class="w-nav-link">Accepted Submissions</a> 
	 <a href="#SC" class="w-nav-link">Committees</a>
        <a href="#" class="nav-link-2 w-nav-link">Sponsors</a> 
	      <a href="#contact" class="w-nav-link">Contact</a>
      </nav>
      <div class="w-nav-button">
        <div class="w-icon-nav-menu"></div>
      </div>
    </div>
  </div>
  <div class="body-container w-container"></div>
  <div class="header-background">
    <div class="headline-container w-container">
      <h1 id="headline" class="heading">The 2022 ICML Workshop on Computational Biology</h1>
    </div>
  </div>
  <main id="main-content" class="header2">
    <div class="container w-container">
      <ul>
	      
<!--	<li class="list-item-4"><strong class="bold-text-2">WCB application has been accepted by ICML &#127881;</strong></li><br>
	 
	<li class="list-item-4"><strong class="bold-text-2">Awaiting approval </strong></li>     
        <li class="list-item-4"><strong class="bold-text-2">TBD, 2022</strong></li> -->
	<li class="list-item-3"><strong class="bold-text">July 22nd, 8:30AM-5:00PM EST, Friday,2022</strong></li>

        <li class="list-item-3"><strong class="bold-text">Room 310, The Baltimore Convention Center, Baltimore, Maryland, USA</strong></li>
	<li class="list-item-7"><strong class="bold-text"> <a href="https://icml.cc/virtual/2022/workshop/13464" target="_blank">Link for virtual</a> </strong></li>     

<!--         <li class="list-item-5"><strong class="bold-text">Workshop date</strong></li> -->
<!--         <li class="list-item-7"><strong class="bold-text">Deadline:July 16th, 2022</strong></li> -->
	 <li class="list-item-7"><strong class="bold-text"> <a href="https://icml.cc/Register/view-registration" target="_blank">Registration</a> </strong></li>     
   
<!-- 	    <ul>
	      <li class="list-item-7"><strong class="bold-text">Reviewer deadline: July 16th 2022 (11:59 PM Pacific Time)</strong></li>
<!--               <li class="list-item-7"><strong class="bold-text">Edit submissions: May 16th, 2022 (11:59 PM Pacific Time)</strong></li> -->
<!--             </ul> -->
   <!-- <li class="list-item-6"><strong class="bold-text">Fellowship application (until July 20th): <a href="https://forms.gle/dfFcXHreGRAoohjx8" target="_blank">Apply</a></strong></li>
  -->        
 <!--        <li class="list-item-3"><strong class="bold-text"><a href="https://icml.cc/virtual/2022/workshop/8366" target="_blank">Click here for Workshop Livestream </a></strong> <br><br></li> -->
<!--        </ul> -->
    </div>
  </main>
  <main id="about" class="header2">
    <div class="w-container">
      <div class="rich-text-block w-richtext">
        <p>Machine learning advances are used in self-driving cars, speech recognition systems, and translation software. However, the COVID-19 pandemic has highlighted the urgency of translating such advances to the domain of biomedicine. Such a pivot requires new machine learning methods to build long-term vaccines and therapeutic strategies, predict immune avoidance, and better repurpose small molecules as drugs.</p>
          <p>The objective of the ICML Workshop on Computational Biology (WCB) is to highlight how machine learning approaches can be tailored to making both translational and basic scientific discoveries with biological data. Practitioners at the intersection of computation, machine learning, and biology are in a unique position to frame problems in biomedicine, from drug discovery to vaccination risk scores, and WCB will showcase such recent research. Commodity lab techniques lead to the proliferation of large complex datasets and require new methods to interpret these collections of high-dimensional biological data, such as genetic sequences, cellular features or protein structures and imaging datasets. These data can be used to make new predictions towards clinical response, uncover new biology, or aid in drug discovery.</p>
<p>This workshop aims to bring together interdisciplinary machine learning researchers working in areas such as computational genomics; neuroscience; metabolomics; proteomics; bioinformatics; cheminformatics; pathology; radiology; evolutionary biology; population genomics; phenomics; ecology, cancer biology; causality; representation learning and disentanglement to present recent advances and open questions to the machine learning community. We especially encourage interdisciplinary submissions that might not neatly fit into one of these categories. </p>
      </div>
    </div>
  </main>


<!-- <main id="Awards" class="main-content">
    <div class="container-2 w-container">
      <h4 class="heading-4">Awards</h4>
    </div>
    <div class="w-container">
      <p class="paragraph-2">
	<span>We are pleased to announce the following awards sponsored by: </span>
      	<h4 class="heading-6">Contributed Talk Award</h4>
	<ul>
	</ul>
      	<h4 class="heading-6">Best Spotlight Talk</h4>
	<ul>
	</ul>
	<h4 class="heading-6">Best Poster Award</h4>
	<ul>
	</ul>
    </div>
  </main>   
 -->


   <main id="Important time" class="main-content">
    <div class="container-2 w-container">
      <h4 class="heading-4">Important Dates</h4>
    </div>
    <div class="w-container">
      <p class="paragraph-2"><span>
        </span>
        Deadline for paper submissions: May <s>8th</s> 13th 2022 (11:59 PM Pacific Time)<br>
	Deadline for edit submissions: May 16th, 2022 (11:59 PM Pacific Time)<br>
        Reviewer deadline: May 30th 2022 <br>
        Notification of acceptance: June 13th, 2022<br>
<!--         Video recording deadline: June 16th, 2022<br> -->
        Camera-ready deadline: July 16th, 2022<br>
        Workshop date: July 22nd, 2022
    </div>
  </main>   
  
  
 <section id="cfp" class="header2">
    <div class="w-container">
      <h4 class="heading-2"> Call for submissions</h4>
      <p class="paragraph-6">We invite extended abstracts dealing with novel algorithms and computational approaches that are robust, 
	      scalable to high-dimensional data, and provide interpretable models of biological systems. These can be applications of ML methods or 
	      bioinformatics approaches to biological and biomedical data or novel approaches that enable new analyses. Papers will be presented in poster 
	      format and some will be selected for oral presentation (contributed and spotlight talks). 
	      Through invited talks and presentations by the participants, this workshop will bring together current advances in Computational Biology and set 
	      the stage for continuing interdisciplinary research discussions.

        
<br><strong>Submission</strong>
 <br>All novel Computational Biology approaches are of interest to the workshop. We welcome original abstracts on preliminary ideas and findings in the following format:<br>
     <ul>
	<li><strong>Extended abstract not exceeding 4 pages</strong> in length (plus 1 optional page for references)</li>
     </ul>
All submissions must use the ICML WCB template with <a href="https://github.com/icml-compbio/icml-compbio.github.io/raw/master/2022/icml_wcb_2022.zip" target="_blank">this sty file</a>. 
The submission does not need to be anonymized. 
<br>Submissions should be made through the <a href="https://cmt3.research.microsoft.com/WCB2022" target="_blank">Microsoft CMT submission portal</a>. 
The submission process is open until <strong>May <s>8th</s> 13th, 2022 (11:59 PM Pacific Time)</strong>. 
For more information about the submission process using CMT, please check this <a href="https://cmt3.research.microsoft.com/docs/help/author/author-submission-form.html" target="_blank">HOW-TO</a>.
<br>Accepted submissions will have the option of being published on the workshop website.
For authors who do not wish their papers to be posted online or become citable, please mention this in the workshop submission.<br>

<br><strong>Instructions for revised submission</strong><br>
In your camera-ready submission please:<br>
1) Change the command in the .sty file at lines 120 -123 to <br>
\newcommand{\ICML@appearing}{The 2022 ICML Workshop on Computational Biology. Copyright 2022 by the author(s).} <br>

2) Comment \usepackage{icml2022} in the main .tex file and uncomment \usepackage[accepted]{icml2022}.<br>
Revised submissions should be made through the CMT by July 16th.


<br><br><strong>Poster</strong>
<br>This year we only doing in-person poster presentation. The size for poster is recommended to be 24"w x 36"h or smaller than 36"w x 48"h. 
 <br>

<!-- <li>Thumbnail Must be 320 width x 256 height PNG </li>
We are currently working with the ICML team to finish setting up the submission portal for the poster files and thumbnails, which is expected to be open for your submissions in the coming days. 
When this is complete, you will be able to submit your files using the following link: <a href="https://icml.cc/Conferences/2022/PosterUpload">https://icml.cc/Conferences/2022/PosterUpload</a>.<br> 
Please ensure you are logged in to the ICML account used for registration when submitting your poster.
We will post information on our workshop website when the submission system is open.
	      
<br><br><strong>Awards</strong>
<br>All accepted contributions shall be presented at the virtual poster session. There will be Awards for Best Poster Presentations. In addition, a set of best submissions will also have the opportunity to present their work as Contributed Talks and receive awards.
<br><br><strong>Registration</strong><br>All participants must register for the Workshop through the <a href="https://icml.cc/Register/view-registration" target="_blank">ICML 2022 conference</a>.
<br><br><strong>Contact</strong><br>
For workshop-related queries please contact:<a href="mailto:workshopcompbio@gmail.com">workshopcompbio@gmail.com</a></p>
  -->
   
 
   </div>
  </section> 
  
<!--   schedule  -->
	  
<!--  <section id="schedule" class="header2">
    <div class="w-container">
      <h4 class="heading-2">Schedule</h4>  
	 <div class="flex-container w-container"><img src="2022/img/schedule_jul13.png" width="1000" alt="" class="image-3">
-->
	  

  <section id="schedule" class="header2">
    <div class="w-container">
      <h4 class="heading-2">Schedule</h4>
      <p class="paragraph-6">
        <!--<a href="">Click here for Workshop Livestream </a> <br><br>-->
        
        * Times below are in EDT <br>
	     
	<style type="text/css">
.tg  {border-collapse:collapse;border-spacing:0;}
.tg td{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
  overflow:hidden;padding:10px 5px;word-break:normal;}
.tg th{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
  font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;}
.tg .tg-ycr8{background-color:#ffffff;text-align:left;vertical-align:top}
@media screen and (max-width: 767px) {.tg {width: auto !important;}.tg col {width: auto !important;}.tg-wrap {overflow-x: auto;-webkit-overflow-scrolling: touch;}}</style>
	     
<div class="tg-wrap"><table class="tg">
<tbody>
  <tr>
    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">8:30 - 8:40 am </span></td>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Opening Remarks (Cassandra Burdziak)</span></td>
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="4"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#434343">Session 1 (Chair: Cassandra Burdziak)</span></td>
  </tr>
  <tr>
    <td class="tg-ycr8" rowspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">8:40 - 9:20 am </span></td>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Tamara Broderick </span></td>
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Predicting and maximizing genomic variant discovery via Bayesian nonparametrics</span></td>
    <!-- <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
  </tr>
  <tr>
    <td class="tg-ycr8" rowspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">9:20 - 9:45 am</span></td>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Damiano Sgarbossa</span></td>
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Generative power of a protein language model trained on multiple sequence alignments</span></td>
    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2022/papers/WCBICML2022_paper_6.pdf" target="_blank">Paper</a>]</span></td>
    <!--<td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td>  -->
  </tr>
  <tr>
    <td class="tg-ycr8" rowspan="4"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">9:45 - 10:00 am</span></td>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Spotlight talks</span></td>
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Daniel Hesslow - RITA: a study on scaling up generative protein sequence models</span></td>
    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2022/papers/WCBICML2022_paper_12.pdf" target="_blank">Paper</a>]</span></td>
    <!--<td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Amir Alavi - Learning batch-invariant representations with domain adaptation in large scale proteomics data</span></td>
  <!--    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2022/papers/WCBICML2022_paper_.pdf" target="_blank">Paper</a>]</span></td>
    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Haiyi Mao & Minxue Jia - COEM: cross-modal embedding for metacell identification</span></td>
    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2022/papers/WCBICML2022_paper_17.pdf" target="_blank">Paper</a>]</span></td>
    <!--<td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
  </tr>
  <tr>
    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">10:00 - 10:30 am </span></td>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Coffee break</span></td>
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="4"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#434343">Session 2 (Chair: Cameron Park)</span></td>
  </tr>
  <tr>
    <td class="tg-ycr8" rowspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">10:30 - 11:10 am</span></td>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Jean Fan</span></td>
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Towards a common coordinate framework: alignment of spatially resolved omics data</span></td>
   <!--  <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Live talk</span></td> -->
  </tr>
  <tr>
    <td class="tg-ycr8" rowspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">11:10 - 11:35 am</span></td>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Haoran Zhang</span></td>
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">BayesTME: a reference-free Bayesian method for end-to-end analysis of spatial transcriptomic data</span></td>
  <!--  <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2022/papers/WCBICML2022_paper_.pdf" target="_blank">Paper</a>]</span></td>
    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
  </tr>
  <tr>
    <td class="tg-ycr8" rowspan="6"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">11:35 - 12:00 pm </span></td>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Spotlight talks</span></td>
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Hannes Stärk - EquiBind: geometric deep learning for drug binding structure prediction</span></td>
 <!--    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2022/papers/WCBICML2022_paper_.pdf" target="_blank">Paper</a>]</span></td>
    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Sophie Jaro - Learning to rank metabolites across datasets</span></td>
<!--     <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2022/papers/WCBICML2022_paper_.pdf" target="_blank">Paper</a>]</span></td>
    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">James Zou - 7-UP: generating in silico CODEX from a small set of immunofluorescence markers</span></td>
    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2022/papers/WCBICML2022_paper_53.pdf" target="_blank">Paper</a>]</span></td>
    <!--<td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Andrew J. Jung - RTfold: RNA secondary structure prediction using deep learning with domain inductive bias</span></td>
    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2022/papers/WCBICML2022_paper_38.pdf" target="_blank">Paper</a>]</span></td>
    <!--<td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Haotian Cui (STREAMED) - A deep learning framework for estimating cell-specific kinetic rates of RNA velocity</span></td>
<!--     <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2022/papers/WCBICML2022_paper_.pdf" target="_blank">Paper</a>]</span></td>
    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
  </tr>
  <tr>
    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">12:00 - 1:30pm   </span></td>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000"> <a href="#papers" class="w-nav-link">Poster session 1</a> <!-- [<a href="https://eventhosts.gather.town/oTvLwqGTGVzInPP8/compbio-w-poster-room-1" target="_blank">Virtual World Room 1 </a>] --></span></td>
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="4"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#434343">Session 3 (Chair: Pascal Notin)</span></td>
  </tr>
  <tr>
    <td class="tg-ycr8" rowspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">1:30 - 2:20 pm</span></td>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Panel discussion: Mohammad AlQuraishi, Elana Fertig, Patrick Schwab, Neeha Zaidi</span></td>
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">ML that matters: Discovering new drugs and treatments for challenging diseases using AI</span></td>
   <!-- <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
  </tr>
  <tr>
    <td class="tg-ycr8" rowspan="9"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">2:20 - 3:00 pm</span></td>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Spotlight talks </span></td>
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Achille Nazaret - Probabilistic basis decomposition for characterizing temporal dynamics of gene expression</span></td>
     <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2022/papers/WCBICML2022_paper_33.pdf" target="_blank">Paper</a>]</span></td>
    <!--<td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Kieran Elmes - SNVformer: an attention-based deep neural network for GWAS data</span></td>
     <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2022/papers/WCBICML2022_paper_58.pdf" target="_blank">Paper</a>]</span></td>
    <!--<td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Christopher Hendra - Extracting part of signal representation from direct RNA squiggle for modification detection</span></td>
   <!--  <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2022/papers/WCBICML2022_paper_.pdf" target="_blank">Paper</a>]</span></td>
    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Hagai M Kariti - A mechanistic probabilistic model of genomic compartments</span></td>
  <!--   <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2022/papers/WCBICML2022_paper_.pdf" target="_blank">Paper</a>]</span></td>
    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Kevin Wu - TCR-BERT: learning the grammar of T-cell receptors for flexible antigen-binding analyses</span></td>
    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2022/papers/WCBICML2022_paper_65.pdf" target="_blank">Paper</a>]</span></span></td>
    <!--<td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Yu Li - Interpretable RNA foundation model from unannotated data for highly accurate RNA structure and function predictions</span></td>
   <!--  <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Paper</span></td>
    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Pooja Kathail - Assessing the utility of genomic deep learning models for disease-relevant variant effect prediction</span></td>
     <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">[<a href="2022/papers/WCBICML2022_paper_71.pdf" target="_blank">Paper</a>]</span></td>
    <!--<td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Yanan Long - Molecular fingerprints are a simple yet effective solution to the drug–drug interaction problem</span></td>
   <!--  <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Paper</span></td>
    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
  </tr>
  <tr>
    <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">3:00 - 4:30 pm   </span></td>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000"><a href="#papers" class="w-nav-link">Poster session 2</a> <!-- [<a href="https://eventhosts.gather.town/lSxhSe2i0AEPMotS/compbio-w-poster-room-3" target="_blank">Virtual World Room 3 </a>] --> </span></td>
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="4"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#434343">Session 4 (Chair: Yubin Xie)</span></td>
  </tr>
  <tr>
    <td class="tg-ycr8" rowspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">4:30 - 4:55 pm </span></td>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Cameron Park</span></td>
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">DIISCO: dynamic intercellular interactions in single cell transcriptomics</span></td>
     <!--  <td class="tg-ycr8"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">Live talk</span></td> -->
  </tr>
  <tr>
    <td class="tg-ycr8" rowspan="2"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000">4:55 - 5:20 pm</span></td>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Zhenqin Wu</span></td>
  </tr>
  <tr>
    <td class="tg-ycr8" colspan="3"><span style="font-style:normal;text-decoration:none;color:#000">SPACE-GM: geometric deep learning of disease-associated microenvironments from multiplex spatial protein profiles</span></td>
   <!--    <td class="tg-ycr8"><span style="font-style:normal;text-decoration:none;color:#000">Paper</span></td>
    <td class="tg-ycr8"><span style="font-style:normal;text-decoration:none;color:#000">Slideslive</span></td> -->
  </tr>
  <tr>
    <td class="tg-ycr8"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">5:20 - 5:30 pm   </span></td>
    <td class="tg-ycr8" colspan="3"><span style="font-weight:700;font-style:normal;text-decoration:none;color:#000">Award ceremony and closing remarks (Yubin Xie)</span></td>
  </tr>
</tbody>
</table></div>      
      </p>
    </div>
  </section>
	  

<!--   Speaker list  -->
   <header id="speaker" class="hero">
    <div class="w-container">
      <h4 class="heading-3">Invited speakers</h4>
      <p class="paragraph-10">*Listed alphabetically</p>
    </div>

    <div class="flex-container w-container"><img src="2022/img/head_pic.jpg" width="225" alt="" class="image-3">
      <div class="hero-image-mask"></div>
      <div class="div-block-4">
        <h5 class="heading-9"><a href="https://tamarabroderick.com" target="_blank">Tamara Broderick</a></h5>
        <p class="paragraph-7">Associate Professor <br> EECS Department at Massachusetts Institute of Technology
 </p>
      </div>
    </div>

    <div class="flex-container w-container"><img src="2022/img/Jean-Fan.jpg" width="225" alt="" class="image-3">
      <div class="hero-image-mask"></div>
      <div class="div-block-4">
        <h5 class="heading-9"><a href="https://jef.works" target="_blank">Jean Fan</a></h5>
        <p class="paragraph-7">Assistant Professor<br> Biomedical Engineering Department at Johns Hopkins University</p>
      </div>
    </div>
	  
	   
	   
	   
	   
   <header id="panelists" class="hero">
    <div class="w-container">
      <h4 class="heading-3">Invited panelists</h4>
      <p class="paragraph-10">*Listed alphabetically</p>
    </div>

    <div class="flex-container w-container"><img src="2022/img/dral.jpg" width="225" alt="" class="image-3">
      <div class="hero-image-mask"></div>
      <div class="div-block-4">
        <h5 class="heading-9"><a href="https://www.aqlab.io" target="_blank">Mohammad AlQuraishi</a></h5>
        <p class="paragraph-7">Assistant Professor <br> Columbia University
 </p>
      </div>
	    
    </div>

    <div class="flex-container w-container"><img src="2022/img/drfertig.jpg" width="225" alt="" class="image-3">
      <div class="hero-image-mask"></div>
      <div class="div-block-4">
        <h5 class="heading-9"><a href="https://fertiglab.com" target="_blank">Elana Fertig</a></h5>
        <p class="paragraph-7">Director of The Research Program in Quantitative Sciences <br>  Johns Hopkins Kimmel Cancer Center
</p>
      </div>
    </div>

	   
	   
    <div class="flex-container w-container"><img src="2022/img/drschwab.png" width="225" alt="" class="image-3">
      <div class="hero-image-mask"></div>
      <div class="div-block-4">
        <h5 class="heading-9"><a href="http://schwabpatrick.com" target="_blank">Patrick Schwab</a></h5>
        <p class="paragraph-7">Director<br> Machine Learning and Artificial Intelligence at GSK</p>
      </div>
    </div>
	   
    <div class="flex-container w-container"><img src="2022/img/drzaidi.jpg" width="225" alt="" class="image-3">
      <div class="hero-image-mask"></div>
      <div class="div-block-4">
        <h5 class="heading-9"><a href="https://www.hopkinsmedicine.org/profiles/details/neeha-zaidi" target="_blank">Neeha Zaidi</a></h5>
        <p class="paragraph-7">Assistant Professor<br> Johns Hopkins Medicine</p>
      </div>
    </div>
	   
	   
  <!-- Accepted papers -->
  <header id="papers" class="hero">
    <div class="w-container">
      <h4 class="heading-3">Accepted Submissions</h4>
      <br>
      <h5 class="heading-4">Poster session 1</h5>
      <p class="paragraph-10" align="justify">
  	
	<ul>

        <li> Damiano Sgarbossa (EPFL); Umberto Lupo (EPFL); Anne-Florence Bitbol (EPFL).
        <strong> Generative power of a protein language model trained on multiple sequence alignments. </strong>
        [<a href="2022/papers/WCBICML2022_paper_6.pdf" target="_blank">paper</a>]
		</li>
	<br>
	
	<li> Haoran Zhang (University of Texas at Austin); Wesley Tansey (Memorial Sloan); Jacqueline Chou (Weill Cornell Medical College).
        <strong> BayesTME: A reference-free Bayesian method for end-to-end analysis of spatial transcriptomic data.</strong> 
       <!-- [<a href="2021/papers/WCBICML2021_paper_.pdf" target="_blank">paper</a>] -->
		</li>
	<br>
	      
	<li> Michail Chatzianastasis (École Polytechnique); Giannis Nikolentzos (Ecole Polytechnique); Michalis Vazirgiannis (École Polytechnique).
        <strong> Mass Enhanced Node Embeddings for Drug Repurposing. </strong> 
        [<a href="2022/papers/WCBICML2022_paper_4.pdf" target="_blank">paper</a>]
		</li>
	<br>
	      
	 <li> Calvin McCarter (Hedgehog Technology).
        <strong> Confounded Domain Adaptation: A Framework for Biological Data Batch Correction. </strong>
       <!-- [<a href="2021/papers/WCBICML2021_paper_.pdf" target="_blank">paper</a>] -->
		 </li>
	<br>
	      
	<li> Asif Khan (University of Edinburgh); Alexander I Cowen-Rivers (Huawei R&D UK); Derrick-Goh-Xin Deik (Huawei Noahs Ark Lab); Antoine F. Grosnit (Huawei); Philippe Robert (Department of Immunology, University of Oslo); Victor Greiff (Department of Immunology, University of Oslo); Kamil Dreczkowski (Huawei); Rasul Tutunov (Huawei); Dany Bou-Ammar (American University of Beirut Medical Centre); Jun Wang (UCL); Haitham Ammar (Huawei).
        <strong> Trust-region Bayesian optimisation with developability constraints enables sample efficient, high-affinity antibody design. </strong>
       <!-- [<a href="2021/papers/WCBICML2021_paper_.pdf" target="_blank">paper</a>] -->
		</li>
	<br>
	      
	<li>  Hayda Almeida (UQAM); Adrian Tsang (Concordia University); Abdoulaye Banire Diallo (UQAM).
        <strong> Reinforcement learning to optimize fungal Biosynthetic Gene Clusters. </strong>
        [<a href="2022/papers/WCBICML2022_paper_11.pdf" target="_blank">paper</a>]
		</li>
	<br>
	      
	<li> Yiqing Shen (Shanghai Jiao Tong University); Bingxin ZHOU (The University of Sydney); Xinye Xiong (SJTU); Ruitian Gao (Shanghai Jiao Tong University); Yuguang Wang (Shanghai Jiao Tong University).
        <strong>How Graph Neural Networks Enhance Convolutional Neural Networks Towards Mining the Topological Structures from Histology.</strong> 
        [<a href="2022/papers/WCBICML2022_paper_16.pdf" target="_blank">paper</a>]
		</li>
	<br>
	      
	<li> Abbas-Ali Heydari (University of California, Merced); Oscar Davalos (University of California, Merced); Katrina Hoyer (University of California, Merced); Suzanne Sindi (University of California, Merced).
        <strong> N-ACT: An Interpretable Deep Learning Model for Automatic Cell Type and Salient Gene Identification. </strong> 
        [<a href="2022/papers/WCBICML2022_paper_18.pdf" target="_blank">paper</a>]
		</li>
	<br>
	      
	<li> Konstantia Georgouli (Lawrence Livermore National Laboratory); Helgi Ingolfur Ingolfsson (Lawrence Livermore National Laboratory); Mark Heimann (Lawrence Livermore); Felice Lightstone (Lawrence Livermore National Laboratory); Peer-Timo Bremer (LLNL); Harsh Bhatia (Lawrence Livermore National Laboratory).
        <strong> Emerging Patterns in the Continuum Representation of Protein-Lipid Fingerprints. </strong> 
       <!-- [<a href="2021/papers/WCBICML2021_paper_.pdf" target="_blank">paper</a>] -->
		</li>
	<br>
	      
	<li> Harsh Bhatia (Lawrence Livermore National Laboratory); Helgi Ingólfsson (Lawrence Livermore National Laboratory); Felice LIghtstone (Lawrence Livermore National Laboratory); Peer-Timo Bremer (LLNL).
        <strong> Capturing Similarity among Simulated Patches of Human Cell Membrane. </strong>
       <!-- [<a href="2021/papers/WCBICML2021_paper_.pdf" target="_blank">paper</a>] -->
		</li>
	<br>
	      
	<li> Fikret Aydin (Lawrence Livermore National Laboratory); Konstantia Georgouli (Lawrence Livermore National Laboratory); Gautham Dharuman (Lawrence Livermore National Laboratory); Jim Glosli (Lawrence Livermore National Laboratory); Felice Lightstone ( Lawrence Livermore National Laboratory); Helgi Ingólfsson (Lawrence Livermore National Laboratory); Peer-Timo Bremer (LLNL); Harsh Bhatia (Lawrence Livermore National Laboratory).
        <strong> Identifying Orientation-specific Lipid-protein Fingerprints using Deep Learning. </strong> 
       <!-- [<a href="2021/papers/WCBICML2021_paper_.pdf" target="_blank">paper</a>] -->
		</li>
	<br>
	      
	<li> Umberto Lupo (EPFL); Damiano Sgarbossa (EPFL); Anne-Florence Bitbol (EPFL).
        <strong> Protein language models trained on multiple sequence alignments learn phylogenetic relationships. </strong>
        [<a href="2022/papers/WCBICML2022_paper_25.pdf" target="_blank">paper</a>]
		</li>
	<br>
	      
	<li> Stephen Y Zhang (University of Melbourne); Michael Stumpf (University of Melbourne).
        <strong> Inferring cell-specific causal regulatory networks from drift and diffusion. </strong> 
        [<a href="2022/papers/WCBICML2022_paper_28.pdf" target="_blank">paper</a>]
		</li>
	<br>
	      
	<li> Dmitrii Gavrilev (Skolkovo Institute of Science and Technology); Nurlybek Amangeldiuly (Skolkovo Institute of Science and Technology); Sergey Ivanov (Criteo); Evgeny Burnaev (Skoltech).
        <strong>High Performance of Gradient Boosting in Binding Affinity Prediction. </strong>
        [<a href="2022/papers/WCBICML2022_paper_31.pdf" target="_blank">paper</a>] 
		</li>
	<br>
	      
	<li> Karl Pichotta (Memorial Sloan Kettering Cancer Center); Wesley Tansey (Memorial Sloan).
        <strong> Zero-Shot Prediction of Drug Combination Activity for High-Throughput Screens. </strong> 
        [<a href="2022/papers/WCBICML2022_paper_32.pdf" target="_blank">paper</a>]
		</li>
	<br>
	      
	<li> Sharrol Bachas (Absci); Goran Rakocevic (AbSci); David Spencer (AbSci); Robel Haile (AbSci); Anand Sastry (Absci); John Sutton (AbSci); George Kasun (AbSci); Christa Kohnert (AbSci); Cailen McCloskey (AbSci); Edriss Yassine (AbSci); Borka Medjo (AbSci); Nebojsa Tijanic (AbSci); Shaheed Abdulhaqq (AbSci); Randal Olsen (AbSci); Jennifer Stanton (AbSci); Bailey White (AbSci); Rebecca Viazzo (AbSci); Rebecca Consbruck (AbSci); Hayley Carter (AbSci); Alexander Brown (AbSci); Joshua Meier (Absci); Matthew Weinstock (AbSci); Gregory Hannum (AbSci); Ariel Schwartz (AbSci); Miles Gander (AbSci); Roberto Spreafico (AbSci).
        <strong> Antibody design by optimization of affinity and naturalness using language models. </strong>
       <!-- [<a href="2021/papers/WCBICML2021_paper_.pdf" target="_blank">paper</a>] -->
		</li>
	<br>
	      
	<li> James Leiner (Carnegie Mellon University); Brian Manzo (University of Michigan); Aaditya Ramdas (Carnegie Mellon University); Wesley Tansey (Memorial Sloan).
        <strong> Scalable Causal Structure Learning via Amortized Conditional Randomization Testing. </strong>
       <!-- [<a href="2021/papers/WCBICML2021_paper_.pdf" target="_blank">paper</a>] -->
		</li>
	<br>
	      
	<li> Casey Bradshaw (Columbia University); Wesley Tansey (Memorial Sloan).
        <strong> Identifying Spatial Biomarkers from Cellular Imaging Data. </strong>
       <!-- [<a href="2021/papers/WCBICML2021_paper_.pdf" target="_blank">paper</a>] -->
		</li>
	<br>
	      
	<li> Daniel Hesslow (LightOn); Niccolò Zanichelli (University of Parma); Pascal Notin (University of Oxford); Iacopo Poli (LightOn); Debora Marks (Harvard).
        <strong> RITA: a Study on Scaling Up Generative Protein Sequence Models. </strong>
        [<a href="2022/papers/WCBICML2022_paper_12.pdf" target="_blank">paper</a>]
		</li>
	<br>
	      
	<li> Amir Alavi (Seer Inc); Harendra Guturu (Seer inc); Mahdi Zamanighomi (Seer Inc); Tristan Brown (Seer Inc); Jian Wang (Seer Inc); Sam Cutler (Seer Inc); Biao Li (Seer Inc); Khatereh Motamedchaboki (Seer Inc); Asim Siddiqui (Seer Inc); Serafim Batzoglou (Seer Inc).
        <strong> Learning Batch-Invariant Representations with Domain Adaptation in Large Scale Proteomics Data. </strong> 
       <!-- [<a href="2021/papers/WCBICML2021_paper_.pdf" target="_blank">paper</a>] -->
		</li>
	<br>
	      
	<li> Haiyi Mao (University of Pittsburgh); Minxue Jia (University of Pittsburgh); Xiaotian Dou (University of Pittsburgh); Haotian Zhang (University of PIttsburgh); Panayiotis Benos (University of Pittsburgh).
        <strong> COEM: Cross-Modal Embedding for MetaCell Identification. </strong>
        [<a href="2022/papers/WCBICML2022_paper_17.pdf" target="_blank">paper</a>]
		</li>
	<br>
	      
	<li> Hannes Stärk (Massachusetts Institute of Technology); Octavian Ganea (MIT); Lagnajit Pattanaik (MIT); Dr.Regina Barzilay (Massachusetts institute of technology); Tommi Jaakkola (MIT).
        <strong> EquiBind: Geometric Deep Learning for Drug Binding Structure Prediction. </strong>
       <!-- [<a href="2021/papers/WCBICML2021_paper_.pdf" target="_blank">paper</a>] -->
		</li>
	<br>
	      
	<li> Haotian Cui (University of Toronto); Hassaan Maan (University of Toronto); Michael Taylor (University of Toronto); Bo Wang.
        <strong> A Deep Learning Framework for Estimating Cell-specific Kinetic Rates of RNA Velocity. </strong> 
        [<a href="2022/papers/WCBICML2022_paper_24.pdf" target="_blank">paper</a>] 
		</li>
	<br>
	      
	<li> Sophie Jaro (Memorial Sloan Kettering); Benji Freeman (Memorial Sloan Kettering); Tricia Park (Memorial Sloan Kettering); Sam Keene (The Cooper Union); Ed Reznik (Memorial Sloan Kettering Cancer Center); Wesley Tansey (Memorial Sloan).
        <strong> Learning to rank metabolites across datasets. </strong>
       <!-- [<a href="2021/papers/WCBICML2021_paper_.pdf" target="_blank">paper</a>] -->
		</li>
	<br>
	      
	<li> Andrew J Jung (University of Toronto); Leo J Lee (University of Toronto); Jiexin Gao; Brendan Frey (U. Toronto).
        <strong>RTfold: RNA secondary structure prediction using deep learning with domain inductive bias. </strong> 
       [<a href="2022/papers/WCBICML2022_paper_38.pdf" target="_blank">paper</a>]
		</li>
	<br>
	      
	<li> Eric Wu (Stanford University); Alexandro Trevino (Enable Medicine); Zhenqin Wu (Stanford University); Kyle Swanson (Stanford University); Honesty Kim (Enable Medicine); H. Blaize D’Angio (Enable Medicine); Ryan Preska (Enable Medicine); Gregory Charville (Stanford University); Piero Dalerba (Columbia University); Umamaheswar Duvvuri (University of Pittsburgh); A. Dimitrios Colevas (Stanford University); Aaron Mayer (Enable Medicine); James Zou (Stanford University).
        <strong>7-UP: generating in silico CODEX from a small set of immunofluorescence markers.</strong> 
        [<a href="2022/papers/WCBICML2022_paper_53.pdf" target="_blank">paper</a>]
		</li>
	<br>
		
	</ul>

      <h5 class="heading-4">Poster session 2</h5>
      <p class="paragraph-10" align="justify">

	<ul>
	<li> Cameron Park (Columbia University); Shouvik Mani (Columbia University); Elham Azizi (Columbia University).
        <strong>DIISCO: Dynamic Intercellular Interactions in Single Cell transcriptOmics. </strong>
       <!-- [<a href="2021/papers/WCBICML2021_paper_.pdf" target="_blank">paper</a>] -->
		</li>
	<br>
	      
	<li> Zhenqin Wu (Stanford University); Alexandro Trevino (Enable Medicine); Eric Wu (Stanford University); Kyle Swanson (Stanford University); Honesty Kim (Enable Medicine); H. Blaize D’Angio (Enable Medicine); Ryan Preska (Enable Medicine); Gregory Charville (Stanford University); Piero Dalerba (Columbia University); Ann Marie Egloff (Brigham and Women’s Hospital); Ravindra Uppaluri (Dana-Farber Cancer Institute); Umamaheswar Duvvuri (University of Pittsburgh); Aaron Mayer (Enable Medicine); James Zou (Stanford University).
        <strong> SPACE-GM: geometric deep learning of disease-associated microenvironments from multiplex spatial protein profiles. </strong> 
       <!-- [<a href="2021/papers/WCBICML2021_paper_.pdf" target="_blank">paper</a>] -->
		</li>
	<br>
	      
	<li> Prashnna K Gyawali (Stanford University); Yann Guen (Stanford University); Xiaoxia Liu (Stanford University); Hua Tang (Stanford University); James Zou (Stanford University); Zihuai He (Stanford University).
        <strong> Disentangling ancestry representations for improving genetic risk prediction across diverse population.</strong> 
       <!-- [<a href="2021/papers/WCBICML2021_paper_.pdf" target="_blank">paper</a>] -->
		</li>
	<br>
	      
	<li> Shounak Chattopadhyay (Duke University); Rungang Han (Duke University); David Dunson (Duke University); Wesley Tansey (Memorial Sloan).
        <strong> Hierarchically weighted mixture modeling of drug combination responses. </strong>
       <!-- [<a href="2021/papers/WCBICML2021_paper_.pdf" target="_blank">paper</a>] -->
		</li>
	<br>
	      
	<li> Jan Engelmann (Helmholtz Munich); Leon Hetzel (Helmholtz Munich, TUM); Giovanni Palla (Helmholtz Munich, TUM); Lisa Sikkema (Helmholtz Munich, TUM); Malte Luecken (Helmholtz Munich); Fabian Theis.
        <strong>Uncertainty Quantification for Atlas-Level Cell Type Transfer. </strong>
        [<a href="2022/papers/WCBICML2022_paper_44.pdf" target="_blank">paper</a>]
		</li>
	<br>
	      
	<li> Michael Murphy (MIT); Kevin Yang (Microsoft); Stefanie Jegelka (MIT); Ernest Fraenkel (MIT).
        <strong> Learning representations from mass spectra for peptide property prediction. </strong> 
        [<a href="2022/papers/WCBICML2022_paper_45.pdf" target="_blank">paper</a>] 
		</li>
	<br>
	      
	<li> Lechuan Li (Rice University); Ruth Dannenfelser (Rice University); Yu Zhu (Rice University); Nathaniel Hejduk (Rice University); Santiago Segarra (Rice University); Vicky Yao (Rice University).
        <strong>Joint Embedding of Biological Networks for Cross-Species Functional Alignment. </strong> 
       <!-- [<a href="2021/papers/WCBICML2021_paper_.pdf" target="_blank">paper</a>] -->
		</li>
	<br>
	      
	<li> Kemal Inecik (Institute of Computational Biology, Helmholtz Center); Andreas Uhlmann (Institute of Computational Biology, Helmholtz Center); mohammad lotfollahi (Helmholtz Zentrum München); Fabian Theis.
        <strong> MultiCPA: Multimodal Compositional Perturbation Autoencoder. </strong>
        [<a href="2022/papers/WCBICML2022_paper_50.pdf" target="_blank">paper</a>] 
		</li>
	<br>
	      
	<li> Peter D Tonner (National Institute of Standard and Technology); Abe Pressman (National Institute of Standards and Technology); David Ross (National Institute of Standards and Technology).
        <strong> LANTERN: An Interpretable Genotype-Phenotype Landscape Model.  </strong>
       <!-- [<a href="2021/papers/WCBICML2021_paper_.pdf" target="_blank">paper</a>] -->
		</li>
	<br>
	      
	<li> Amine M. Remita (Université du Québec à Montréal); Abdoulaye Banire Diallo (Université du Québec à Montréal).
        <strong> Toward Inferring Ancestral States and Evolutionary Parameters using a Variational Generative Model for Multiple Sequence Alignments.</strong>
        [<a href="2022/papers/WCBICML2022_paper_54.pdf" target="_blank">paper</a>]
		</li>
	<br>
	      
	<li> Vincent Zaballa (University of California Irvine); Elliot Hui (University of California Irvine).
        <strong> An Optimal Likelihood Free Method for Biological Model Selection.</strong> 
        [<a href="2022/papers/WCBICML2022_paper_55.pdf" target="_blank">paper</a>] 
		</li>
	<br>
	      
	<li> Andrea Karlova (UCL); Wim Dehaen (VSCHT); Andrei Penciu (April19 Discovery Inc); Richard Dallaway (April19 Discovery Inc); Suran Goonatilake (April19 Discovery Inc).
        <strong> PEPSI: Post-docking Evaluation with Protein-Small Molecules Interaction. </strong> 
        [<a href="2022/papers/WCBICML2022_paper_57.pdf" target="_blank">paper</a>]
		</li>
	<br>
	      
	<li> Gary Yao (Mayo Clinic).
        <strong>Combating Extreme Adversarial Vulnerabilities in WSI Classification.</strong> 
        [<a href="2022/papers/WCBICML2022_paper_61.pdf" target="_blank">paper</a>]
		</li>
	<br>
	      
	 <li>David Liu (Stanford University); Ligia Melo (Stanford University); Allan Costa (Center for Bits and Atoms); Martin Vögele (Stanford University); Raphael Townshend (Stanford University); Ron Dror (Stanford University).
	 <strong>Euclidean Transformers for Macromolecular Structures: Lessons Learned. </strong>
        [<a href="2022/papers/WCBICML2022_paper_63.pdf" target="_blank">paper</a>]
		 </li>
	<br>
	      
	<li> Robert Palovics (Stanford University); Tony Wyss-Coray (Stanford University); Baharan Mirzasoleiman (UCLA).
        <strong>Purification of single-cell transcriptomics data with coreset selection.</strong> 
        [<a href="2022/papers/WCBICML2022_paper_67.pdf" target="_blank">paper</a>]
		</li>
	<br>
	      
	 <li> Soroor Hediyeh-zadeh (Institute of Computational Biology); Mohammad Lotfollahi (Helmholtz Zentrum München); Fabian Theis.
        <strong>Continual single-cell architecture surgery for reference mapping. </strong>
        [<a href="2022/papers/WCBICML2022_paper_68.pdf" target="_blank">paper</a>]
		 </li>
	<br>
	      
	 <li> Onuralp Soylemez (Global Blood Therapeutics).
        <strong> Bayesian tensor factorization for predicting clinical outcomes using integrated human genetics evidence.</strong> 
        [<a href="2022/papers/WCBICML2022_paper_73.pdf" target="_blank">paper</a>]
		 </li>
	<br>
	      
	 <li> Achille Nazaret (Columbia University); Joy Fan (Columbia University); Dana Pe'er (Memorial Sloan Kettering Cancer Center); Elham Azizi (Columbia University).
        <strong> Probabilistic basis decomposition for characterizing temporal dynamics of gene expression. </strong> 
        [<a href="2022/papers/WCBICML2022_paper_33.pdf" target="_blank">paper</a>]
		 </li>
	<br>
	      
	<li> Hagai M Kariti (Technion)*; Noam Kaplan (Technion).
        <strong> A mechanistic probabilistic model of genomic compartments. </strong>
       <!-- [<a href="2021/papers/WCBICML2021_paper_.pdf" target="_blank">paper</a>] -->
		</li>
	<br>
	      
       <li> Kieran Elmes (University of Canterbury; University of Otago); Diana Benavides Prado (The University of Auckland); Michael J Witbrock (University of Auckland); Alex Gavryushkin (University of Canterbury).
       <strong> SNVformer: An Attention-based Deep Neural Network for GWAS Data. </strong>
        [<a href="2022/papers/WCBICML2022_paper_58.pdf" target="_blank">paper</a>]
	       </li>
	<br>

	 <li> Shiva Ebrahimi ( University of North Texas ) ; Xuan Guo ( University of North Texas ).
        <strong>Deep Active Learning for De Novo Peptide Sequencing from Data-independent-acquisition Mass Spectrometry.</strong>
        [<a href="2022/papers/WCBICML2022_paper_60.pdf" target="_blank">paper</a>]
		 </li>
	<br>
		
		
	 <li> Christopher Hendra (National University of Singapore); Jonathan Göke (Genomics Institute of Singapore, A*STAR, Singapore); Alex Thiery (National University of Singapore).
        <strong> Extracting Part of Signal Representation from Direct RNA Squiggle for Modification Detection.</strong>
        [<a href="2022/papers/WCBICML2022_paper_62.pdf" target="_blank">paper</a>]
		 </li>
	<br>
		
		

	      
	 <li> Kevin Wu (Stanford); Katie Yost (Stanford University); Bence Daniel (Stanford University); Julia Belk (Stanford University); Yu Xia (Washington University in St. Louis); Takeshi Egawa (Washington University in St. Louis); Ansuman Satpathy (Stanford University); Howard Chang (Stanford University); James Zou (Stanford University).
        <strong>TCR-BERT: learning the grammar of T-cell receptors for flexible antigen-binding analyses.</strong>
        [<a href="2022/papers/WCBICML2022_paper_65.pdf" target="_blank">paper</a>]
	</li>
	<br>
	      
	<li> Jiayang Chen (The Chinese University of Hong Kong); Zhihang Hu (The Chinese University of Hong Kong); Siqi Sun (Fudan University); Qingxiong Tan (Department of Computer Science, Hong Kong Baptist University); Yixuan Wang (Harbin Institute of Technology); Qinze Yu (University of Electronic Science and Technology of China); Licheng Zong (The Chinese University of Hong Kong); Liang Hong (The Chinese University of Hong Kong); Jin Xiao (The Chinese University of Hong Kong); Irwin King (The Chinese University of Hong Kong); Yu Li (The Chinese University of Hong Kong).
        <strong>Interpretable RNA Foundation Model from Unannotated Data for Highly Accurate RNA Structure and Function Predictions. </strong> 
       <!-- [<a href="2021/papers/WCBICML2021_paper_.pdf" target="_blank">paper</a>] -->
	</li>
	<br>
	      
	<li> Pooja Kathail (UC Berkeley)*; Richard W Shuai (UC Berkeley); Ryan Chung (UC Berkeley); Chun Ye (UCSF); Gabriel Loeb (UCSF); Nilah Ioannidis (University of California, Berkeley).
        <strong>Assessing the utility of genomic deep learning models for disease-relevant variant effect prediction. </strong>
        [<a href="2022/papers/WCBICML2022_paper_71.pdf" target="_blank">paper</a>]
	</li>
	<br>
	      
	      
	<li> Yanan Long (University of Chicago); Horace Pan (University of Chicago).
        <strong> Molecular Fingerprints Are a Simple Yet Effective Solution to the Drug–Drug Interaction Problem. </strong> 
        [<a href="2022/papers/WCBICML2022_paper_72.pdf" target="_blank">paper</a>] 
	</li>
	<br>
	 
	</ul>
	      
	      
	     
     
<!--     <div class="flex-container w-container"><img src="images/2022/carola.png" width="225" alt="" class="image-2">
      <div class="hero-image-mask"></div>
      <div class="div-block-6">
        <h5 class="heading-11"><a href="https://www.damtp.cam.ac.uk/user/cbs31/Home.html" target="_blank">Carola-Bibiane Schönlieb</a></h5>
        <p class="paragraph-9">Professor of Applied Mathematics <br> Head of the Cambridge Image Analysis group <br> University of Cambridge <br>
        </p>
      </div>
    </div>
     
    <div class="flex-container w-container"><img src="images/2022/kristin.jpg" width="225" alt="" class="image-2">
      <div class="hero-image-mask"></div>
      <div class="div-block-5">
        <h5 class="heading-10"><a href="https://www.mayo.edu/research/labs/mathematical-neuro-oncology" target="_blank">Kristin Swanson</a></h5>
        <p class="paragraph-8">Vasak and Anna Maria Polak Professor in Cancer Research<br> Vice Chair, Neurosurgery, Mayo Clinic <br></p>

      </div>
    </div> -->

    </p>
      
    </div>
  </header>

  
<!--    Diversity & inclusion  -->
   <main id="DS" class="main-content">
    <div class="container-2 w-container">
      <h4 class="heading-4">Diversity and Inclusion</h4>
    </div>
    <div class="w-container">
      <p class="paragraph-2"><span>
        </span>
	 
	   ICML Workshop on Computational Biology aims to foster an inclusive and welcoming community.    
	   If you have any questions, comments, or concerns, please reach out to workshopcompbio@gmail.com. 
     
	    
        <br>
        We are also featuring other workshops that you might find helpful for diversity and inclusion. 	      
        <a href="https://www.queerinai.com/icml" target="_blank">Queer in AI</a> 
	      
        <br>
	<br>  
	       <strong class="bold-text">Diversity Fellowship </strong>
	<br>  
	We are pleased to announce that we will continue our Diversity Fellowship for students this year. 
        Awards include a free virtual or in-person workshop registration. We encourage applications from underrepresented groups. The deadline is July 15th. Apply <a href="https://forms.gle/HJUBooBYFn4s2sJ67" target="_blank">here</a>.   


    </div>
  </main>   

	
<main id="SC" class="main-content">
    <div class="container-2 w-container">
      <h4 class="heading-4">Steering committee</h4>
    </div>
    <div class="w-container">
      <p class="paragraph-2"><span>
        </span>
	      Dana Pe’er: mskcc.org/research/ski/labs/dana-pe-er<br>
	      Debora Marks: https://www.deboramarkslab.com/<br>
	      Elham Azizi: https://www.azizilab.com/<br>
	      Sandhya Prabhakaran: sandhyaprabhakaran.com <br>
	      Abdoulaye Baniré Diallo: labo.bioinfo.uqam.ca <br>
	      Alexander Anderson: labpages.moffitt.org/andersona/<br>
	      Wesley Tansey: http://wesleytansey.com<br>
	      Julia E. Vogt: mds.inf.ethz.ch/team/detail/julia-vogt/<br>
	</p>
    </div>
  </main>   
  
  <main id="OC" class="main-content">
    <div class="container-2 w-container">
      <h4 class="heading-4">Organizing committee</h4>
    </div>
    <div class="w-container">
      <p class="paragraph-2"><span>
        </span>
	      Yubin Xie: yux2009@med.cornell.edu<br>
	      Cassandra Burdziak: cnb3001@med.cornell.edu<br>
	      Amine Remita: remita.amine@courrier.uqam.ca<br>
	      Mafalda Dias: mafalda_dias@hms.harvad.edu<br>
	      Mauricio Tec: mauriciogtec@utexas.edu<br>
	      Cameron Park: cyp2111@columbia.edu<br>
	      Achille Nazaret: aon2108@columbia.edu<br>
	      Pascal Notin: pascal.notin@cs.ox.ac.uk<br>
	      Steffan Paul: steffanpaul@g.harvard.edu<br>

	    </p> 
    </div>
  </main>


<main id="main-content" class="main-content-2">
    <div class="w-container">
      <div class="rich-text-block-4 w-richtext">
        <h4>Program Committee</h4>
        <p class="paragraph-3">
Abdoulaye Banire Diallo, Université du Québec à Montréal <br>
Achille Nazaret, Columbia University <br>
Adam Gayoso, UC Berkeley <br>
Adrian Heilbut, logphase research <br>
Ahmed Halioui, Mt Intelligent Machines <br>
Alice Bizeul, ETHZ <br>
Amine Remita, Université du Québec à Montréal <br>
Andreas Kirsch, University of Oxford <br>
Anne Siegel, CNRS <br>
Bishnu Sarker, Inria <br>
Bo Yuan, Harvard University <br>
Cameron Park, Columbia University <br>
Cassandra Burdziak, Memorial Sloan Kettering Cancer Center <br>
Chensu Xie, Weill Cornell Medicine <br>
Chetanya Pandya, 2seventy bio <br>
Christopher Garay, Paradigm4 <br>
Debora Marks, Harvard Medical School <br>
Doron Haviv, Cornell <br>
Elham Azizi, Columbia University <br>
Elior Rahmani, UCLA <br>
Elisabetta De Maria, Université Côte d'Azur <br>
Freddie Bickford Smith, University of Oxford <br>
Gilles Gut, ETH Zürich <br>
Han Yuan, Calico Life Sciences <br>
Hayda Almeida, UQAM <br>
Imant Daunhawer, ETH Zurich <br>
Jan Brauner, University of Oxford <br>
Khalil Ouardini, Owkin <br>
Lood van Niekerk, Harvard Medical School <br>
Luis Rueda, University of Windsor <br>
Mafalda Dias, Harvard Medical School <br>
Manu Setty, SKI <br>
Matthew Peterson, Paradigm4 <br>
Mauricio Tec, University of Texas at Austin <br>
Mengting Gu, Visa Research <br>
Mervin Fansler, MSKCC <br>
Mika Jain, Stanford University <br>
Mohammad Lotfollahi, Helmholtz Zentrum München <br>
Nazim Bouatta, Harvard <br>
Neil Band, University of Oxford <br>
Niina Haiminen, IBM T. J. Watson Research Center <br>
Pascal Notin, University of Oxford <br>
Peter Koo, Cold Spring Harbor Laboratory <br>
Ričards Marcinkevičs, ETH Zurich <br>
Romain Lopez, UC Berkeley <br>
Sabeur Aridhi, LORIA <br>
Sandhya Prabhakaran, Moffitt Cancer Center <br>
Sarvesh Nikumbh, Imperial College London and MRC London Institute of Medical Sciences <br>
Sheng Liu, Indiana University School of Medicine <br>
Smita Krishnaswamy, Yale University <br>
Steffan Paul, Harvard Medical School <br>
Thomas Sutter, ETH Zurich <br>
Vianne Gao, Weill Medical College <br>
Victor Greiff, Department of Immunology, University of Oslo <br>
Warith Eddine DJEDDI, FST Manar <br>
Xiang Niu, Memorial Sloan Kettering Cancer Center <br>
Yubin Xie, Memorial Sloan Kettering Cancer Center <br>
Yuelin Shi, California Institute of Technology <br>
        </p>
      </div>
    </div>
  </main>

<section id="sponsor" class="cards-section">
    <div class="w-container">
      <h4 class="heading-4">Sponsors</h4>
    </div>
    <div class="centered-container w-container">

      <div class="cards-grid-container">

        
        <div id="w-node-0d05fd79f6e4-1853825c" class="div-block-2">
          <div class="cards-image-mask"><img src="images/2021/CZ_LOGO.png" alt="" class="cards-image"></div>
          <h6 class="heading-7"><a href="https://chanzuckerberg.com" target="_blank">Chan Zuckerberg Initiative</a></h6>
        </div>
        
        <div id="w-node-0d05fd79f6e4-1853825c" class="div-block-2">
          <div class="cards-image-mask"><img src="images/2021/cancer_dynamics.jpg" alt="" class="cards-image"></div>
          <h6 class="heading-7"><a href="https://cancerdynamics.columbia.edu/" target="_blank">Columbia University Cancer Dynamics</a></h6>
          </div>

        <div id="w-node-0d05fd79f6eb-1853825c" class="div-block">
          <div class="cards-image-mask"><img src="images/2021/jetbrains_logo.png"  alt="" class="cards-image"></div>
          <h6 id="w-node-0d05fd79f6ee-1853825c" class="heading-6"><a href="https://www.jetbrains.com" class="link-3" target="_blank">JetBrains</a></h6>
        </div>
	      

        <div id="w-node-0d05fd79f6eb-1853825c" class="div-block">
          <div class="cards-image-mask"><img src="2022/img/GSK_logo.jpeg" alt="psf" class="cards-image" style="width:256px;height:84px;"></div>
          <h6 class="heading-6"><a href="https://www.gsk.com/en-gb/home/" class="link-3" target="_blank">GSK</a></h6>
        </div>
        
	      
	      
        <div id="w-node-0d05fd79f6eb-1853825c" class="div-block">
          <div class="cards-image-mask"><img src="images/2021/Moffitt-2COLOR_Logo.jpg"  alt="psf" class="cards-image" style="width:256px;height:65px;"></div>
          <h6 class="heading-6"><a href="https://moffitt.org/" class="link-3" target="_blank">Moffitt Cancer Center</a></h6>
        </div>
	      
        <div id="w-node-0d05fd79f6eb-1853825c" class="div-block">
          <div class="cards-image-mask"><img src="2022/img/Nanostring_logo.png"  alt="" class="cards-image" style="width:400px;height:65px;"></div>
          <h6 id="w-node-0d05fd79f6ee-1853825c" class="heading-6"><a href="https://nanostring.com" class="link-3" target="_blank">Nanostring</a></h6>
        </div>

	 <div id="w-node-0d05fd79f6eb-1853825c" class="div-block">
          <div class="cards-image-mask"><img src="2022/img/3.2-SMB-CMYK-black-blue.png"  alt="" class="cards-image" style="width:256px;height:80px;" ></div>
          <h6 id="w-node-0d05fd79f6ee-1853825c" class="heading-6"><a href="https://www.smb.org" class="link-3" target="_blank">The Society for Mathematical Biology</a></h6>
        </div>
        
        <div id="w-node-0d05fd79f6f2-1853825c" class="div-block-3">
          <div class="cards-image-mask"></div>
        </div>
      </div>
    </div>
  </section> 

  
  <main id="main-content" class="main-content-2">
    <div class="w-container">
      <div class="rich-text-block-3 w-richtext">
        <h4>Previous workshops</h4>
        <ul>
          <li><a href="https://icml-compbio.github.io/icml-website-2021/" target="_blank">ICML 2021 Workshop on Computational Biology</a></li>
          <li><a href="https://icml-compbio.github.io/icml-website-2020/" target="_blank">ICML 2020 Workshop on Computational Biology</a></li>
          <li><a href="https://sites.google.com/view/icml-compbio-2019/" target="_blank">ICML 2019 Workshop on Computational Biology</a></li>
          <li><a href="https://sites.google.com/view/wcb2018/" target="_blank">Joint ICML and IJCAI 2018 Workshop on Computational Biology</a></li>
          <li><a href="https://www.google.com/url?q=https%3A%2F%2Fsites.google.com%2Fview%2Fcompbioworkshopicml2017%2Fhome&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNFWzRemcmyQ-NMnfr5nzoYSaBOgcA" target="_blank">ICML 2017 Workshop on Computational Biology</a></li>
          <li><a href="https://www.google.com/url?q=https%3A%2F%2Fsites.google.com%2Fsite%2Fcompbioworkshopicml2016%2F&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNFjnboz8_oyLofUYTQv1Aq_MxW0nA" target="_blank">ICML 2016 Workshop on Computational Biology</a></li>
        </ul>
      </div>
    </div>
  </main>
  <footer id="contact" class="footer">
    <div class="container-3 w-container">
      <h4 class="heading-3">Contact<a href="mailto:workshopcompbio@gmail.com"></a></h4>
    </div>
    <div class="w-container">
      <p class="paragraph-3">For workshop-related queries please contact: <a href="mailto:workshopcompbio@gmail.com?subject=Question%20about%20ICML%202220%20Computational%20Biology%20workshop%20">workshopcompbio@gmail.com</a></p>
    </div>
    <div class="w-container">
      <div class="text-block">Copyright © 2022 ICML compbio workshop. All rights reserved.<br>Designed by Yubin Xie</div>
    </div>
  </footer>
<!--   <script src="https://d3e54v103j8qbb.cloudfront.net/js/jquery-3.4.1.min.220afd743d.js?site=5e9514078c78742fa6aa5dc3" type="text/javascript" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script> -->
  <script src="js/webflow.js" type="text/javascript"></script>
  <!-- [if lte IE 9]><script src="https://cdnjs.cloudflare.com/ajax/libs/placeholders/3.0.2/placeholders.min.js"></script><![endif] -->
</body>
</html>
 <br>
