<!DOCTYPE html>
<!--  This site was created in Webflow. http://www.webflow.com  -->
<!--  Last Published: Wed Apr 22 2020 01:49:40 GMT+0000 (Coordinated Universal Time)  -->
<html data-wf-page="5e951407425f2e221853825c" data-wf-site="5e9514078c78742fa6aa5dc3">
<head>
  <meta charset="utf-8">
  <title>CompBio Workshop ICML 2020</title>
  <meta content="width=device-width, initial-scale=1" name="viewport">
  <meta content="Webflow" name="generator">
  <link href="css/normalize.css" rel="stylesheet" type="text/css">
  <link href="css/webflow.css" rel="stylesheet" type="text/css">
  <link href="css/icml-compbio.webflow.css" rel="stylesheet" type="text/css">
  <!-- [if lt IE 9]><script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.min.js" type="text/javascript"></script><![endif] -->
  <script type="text/javascript">!function(o,c){var n=c.documentElement,t=" w-mod-";n.className+=t+"js",("ontouchstart"in o||o.DocumentTouch&&c instanceof DocumentTouch)&&(n.className+=t+"touch")}(window,document);</script>
  <link href="images/favicon.png" rel="shortcut icon" type="image/x-icon">
  <link href="images/webclip.png" rel="apple-touch-icon">
  <script src="https://code.jquery.com/jquery-1.11.2.min.js"></script>
</head>
<body class="body">
  <div data-collapse="medium" data-animation="default" data-duration="300" class="navbar site-navigation w-nav">
    <div class="navbar-container w-container">
      <a href="index.html" aria-current="page" class="brand-link w-nav-brand w--current">
        <h3 id="site-title" class="site-title">ICML-CompBio-2020</h3>
      </a>
      <div class="menu-button w-nav-button"></div>
    </div>
  </div>
  <div data-collapse="small" data-animation="default" data-duration="50" class="navbar-2 w-nav">
    <div class="w-container">
      <nav role="navigation" class="w-nav-menu"><a href="#cfp" class="w-nav-link"> Call for submissions</a><a href="#schedule" class="w-nav-link"> Schedule</a><a href="#speaker" class="w-nav-link">Invited speakers</a><a href="#papers" class="w-nav-link">Accepted Submissions</a><a href="#OC" class="w-nav-link">Committee</a><a href="#sponsor" class="nav-link-2 w-nav-link">Sponsor</a><a href="#contact" class="w-nav-link">Contact</a></nav>
      <div class="w-nav-button">
        <div class="w-icon-nav-menu"></div>
      </div>
    </div>
  </div>
  <div class="body-container w-container"></div>
  <div class="header-background">
    <div class="headline-container w-container">
      <h1 id="headline" class="heading">The 2020 ICML Workshop on Computational Biology</h1>
    </div>
  </div>
  <main id="main-content" class="header2">
    <div class="container w-container">
      <ul>
        <li class="list-item-4"><strong class="bold-text-2">Date: Fri, July 17, 2020</strong></li>
        <li class="list-item-3"><strong class="bold-text">Location: Virtual conference</strong></li>
         <li class="list-item-3"><strong class="bold-text"><a href="https://icml.cc/virtual/2020/workshop/5721">Click here for Workshop Livestream </a></strong> <br><br></li>
      </ul>
    </div>
  </main>
  <main id="about" class="header2">
    <div class="w-container">
      <div class="rich-text-block w-richtext">
        <p>Computational biology is an interdisciplinary field that develops and applies analytical methods, mathematical and statistical modeling and simulation to analyze and interpret vast collections of biological data, such as genetic sequences, cellular features or protein structures, and imaging datasets to make new predictions towards clinical response, discover new biology or aid drug discovery. The availability of high-dimensional data, at multiple spatial and temporal resolutions has made machine learning and deep learning methods increasingly critical for computational analysis and interpretation of the data. Conversely, biological data has also exposed unique challenges and problems that call for the development of new machine learning methods. </p>
        <p>This workshop aims to bring together researchers working at the unique intersection of Machine Learning and Biology that include areas (and not limited to) such as computational genomics, neuroscience, pathology, radiology, evolutionary biology, population genomics, phenomics, ecology, cancer biology, causality, and representation learning and disentanglement to present recent advances and open questions to the ML community. </p>
      </div>
    </div>
  </main>
  <section id="cfp" class="header2">
    <div class="w-container">
      <h4 class="heading-2"> Call for submissions</h4>
      <p class="paragraph-6">We invite extended abstracts, and highlight papers dealing with novel algorithms and computational approaches that are robust, scalable to high-dimensional data, and provide interpretable models of biological systems. These can be applications of ML methods or bioinformatics approaches to biological and biomedical data or novel approaches that enable new analyses.Papers will be presented in poster format and some will be selected for oral presentation. Through invited talks and presentations by the participants, this workshop will bring together current advances in Computational Biology and set the stage for continuing interdisciplinary research discussions.<br><br><strong>Important Dates</strong><br>Deadline for submissions : May 22nd 2020<br>Reviewer deadline : June 14th 2020 <br>Notification of acceptance : June 17th, 2020<br>Camera-ready deadline : July 10th, 2020<br>Workshop date : July 17th 2020<br><br><strong>Submission</strong><br>All novel Computational Biology approaches are of interest to the workshop. We welcome original abstracts on recently published work as well as preliminary ideas in two different formats:<br>         - <strong>Extended abstracts</strong> <strong>not exceeding 4 pages</strong> in length (plus 1 optional page for references).<br>         - <strong>Highlight papers not exceeding 2 pages</strong> with an abstract and link to recently published paper/code. This avenue can be used to project already published articles.<br>‍<br>All submissions must use the ICML template and <a href="https://drive.google.com/file/d/1S_X24U4-Ztgvnj1eAIfw06tvH4yoKly-/view?usp=sharing" target="_blank">this sty file</a> to correct the footnote. The submission need not be anonymized. If the submission concerns previously published work, please cite the original paper in the workshop submission. <br>Submissions should be made through the <a href="https://www.google.com/url?q=https%3A%2F%2Feasychair.org%2Fconferences%2F%3Fconf%3Dwcbicml2020&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNHvLnJAu2fENz6G3wNZp7a7jYyODg" target="_blank">EasyChair system</a>.<br>Accepted submissions will have the option of being published on the workshop website. For authors who do not wish their papers to be posted online or become citable, please mention this in the workshop submission.<br><br><strong>Instructions for revised submission<br></strong>In your camera-ready submission, please comment \usepackage{icml2020} in the main .tex file and uncomment \usepackage[accepted]{icml2020}. Revised submissions should be made through the <a href="https://www.google.com/url?q=https%3A%2F%2Feasychair.org%2Fconferences%2F%3Fconf%3Dwcbicml2020&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNHvLnJAu2fENz6G3wNZp7a7jYyODg" target="_blank">EasyChair system</a> by July 10th.<br><br><strong>Awards</strong><br>All accepted contributions shall be presented at the virtual poster session. There will be Awards for Best Poster Presentations. In addition, a set of best submissions will also have the opportunity to present their work as Contributed Talks and receive awards.<br><br><strong>Registration</strong><br>All participants must register for the Workshop through the <a href="https://www.google.com/url?q=https%3A%2F%2Ficml.cc%2FRegister%2Fview-registration&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNFgwnJ7SsPdwIslqvkSEyOM6j71ag" target="_blank">ICML 2020 conference</a>.<br><br><strong>Contact</strong><br>For workshop-related queries please contact:<a href="mailto:workshopcompbio@gmail.com">workshopcompbio@gmail.com</a></p>
    </div>
  </section>
  <section id="schedule" class="header2">
    <div class="w-container">
      <h4 class="heading-2"> Schedule</h4>
      <p class="paragraph-6">
        <a href="https://icml.cc/virtual/2020/workshop/5721">Click here for Workshop Livestream </a> <br><br>
        
        * Times below are in EDT
        <h6 class="heading-6">Session 1 (Chair: Cassandra Burdziak)</h6>
      8:50 - 9 am &emsp; Opening Remarks: Dana Pe'er <br> <br>
      
      9 - 9:30 am &emsp; Invited Talk 1: <b> Fabian Theis: </b> Latent space learning and data integration in single cell genomics
      [<a href="https://slideslive.com/38930607/latent-space-learning-and-data-integration-in-single-cell-genomics" target="_blank">slideslive</a>] <br> <br>
        
      9:30 - 9:50 am &emsp; Contributed Talk 1: Jacob C. Kimmel: scNym: Semi-supervised adversarial neural networks for single cell classification
      [<a href="2020/papers/WCBICML2020_paper_44.pdf" target="_blank">paper&nbsp;44</a>]
      [<a href="https://slideslive.com/38930608/scnym-semisupervised-adversarial-neural-networks-for-single-cell-classification" target="_blank">slideslive</a>] <br><br>
        
      9:50-10:15 am  &emsp;Spotlights<br>
&emsp; Neha Prasad: Optimal Transport using GANs for Lineage Tracing
      [<a href="2020/papers/WCBICML2020_paper_8.pdf" target="_blank">paper&nbsp;8</a>]
      [<a href="https://slideslive.com/38931306/superot-optimal-transport-methodology-for-biological-lineage-tracing" target="_blank">slideslive</a>] <br>

&emsp; Lingfei Wang: Normalisr: inferring single-cell differential and co-expression with linear association testing
      [<a href="https://slideslive.com/38931307/normalisr-inferring-singlecell-differential-and-coexpression-with-linear-association-testing" target="_blank">slideslive</a>] <br>

&emsp; Kexin Huang: scGNN: scRNA-seq Dropout Imputation via Induced Hierarchical Cell Similarity Graph
      [<a href="2020/papers/WCBICML2020_paper_17.pdf" target="_blank">paper&nbsp;17</a>]
      [<a href="https://slideslive.com/38931308/scgnn-scrnaseq-dropout-imputation-via-induced-hierarchical-cell-similarity-graph" target="_blank">slideslive</a>] <br>

&emsp; Leander Dony: Variational autoencoders with flexible priors enable robust distribution learning on single-cell RNA sequencing data
      [<a href="2020/papers/WCBICML2020_paper_37.pdf" target="_blank">paper&nbsp;37</a>]
      [<a href="https://slideslive.com/38931309/variational-autoencoders-with-flexible-priors-enable-robust-distribution-learning-on-singlecell-rna-sequencing-data" target="_blank">slideslive</a>] <br>

&emsp; Mohammad Lotfollahi: Out-of-distribution prediction with disentangled representations for single-cell RNA sequencing data
      [<a href="2020/papers/WCBICML2020_paper_41.pdf" target="_blank">paper&nbsp;41</a>]
      [<a href="https://slideslive.com/38931310/outofdistribution-prediction-with-disentangled-representations-for-scrnaseq-data" target="_blank">slideslive</a>] <br> 

      10:15-10:30 am &emsp; Break<br> <br>

        <h6 class="heading-6">Session 2 (Chair: Wesley Tansey)</h6>

      10:30-11 am  &emsp; Invited Talk 2:<b> Olga Troyanskaya: </b> Decoding the genome with AI
      [<a href="https://slideslive.com/38930901/decoding-the-genome-with-ai" target="_blank">slideslive</a>]
 <br> <br>

      11 - 11:20am  &emsp; Contributed Talk 2: Geoffrey Fudenberg: Predicting 3D genome folding from DNA sequence
      [<a href="2020/papers/WCBICML2020_paper_20.pdf" target="_blank">paper&nbsp;20</a>]
      [<a href="https://slideslive.com/38930902/predicting-3d-genome-folding-from-dna-sequence" target="_blank">slideslive</a>] <br> <br>

      11:20 - 11:55 am &emsp; Paper highlights:<br>

&emsp;	Sanjiv Kumar Dwivedi: Deriving Disease Modules from the Compressed Transcriptional Space Embedded in a Deep Autoencoder
      [<a href="2020/papers/WCBICML2020_paper_3.pdf" target="_blank">paper&nbsp;3</a>]
      [<a href="https://slideslive.com/38931311/deriving-disease-modules-from-the-compressed-transcriptional-space-embedded-in-a-deep-autoencoder" target="_blank">slideslive</a>] <br>

&emsp;	Iman Deznaby: DeepKinZero: Zero-Shot Learning for Predicting Kinase-Phosphosite Associations
      [<a href="2020/papers/WCBICML2020_paper_4.pdf" target="_blank">paper&nbsp;4</a>]
      [<a href="https://slideslive.com/38931312/deepkinzero-zeroshot-learning-for-predicting-kinasephosphosite-associations-involving-understudied-kinases" target="_blank">slideslive</a>] <br>

&emsp;	Christian Matek: A publicly available database for developing machine learning applications to differentiate leukocytes and recognise malignant cells in peripheral blood
      [<a href="2020/papers/WCBICML2020_paper_14.pdf" target="_blank">paper&nbsp;14</a>]
      [<a href="https://slideslive.com/38931313/a-publicly-available-dataset-for-developing-machine-learning-applications-to-differentiate-leukocytes-and-recognise-malignant-cells-in-peripheral-blood" target="_blank">slideslive</a>] <br>

&emsp;	Serghei Mangul: Profiling immunoglobulin repertoires across multiple human tissues using RNA Sequencing
      [<a href="2020/papers/WCBICML2020_paper_21.pdf" target="_blank">paper&nbsp;21</a>]
      [<a href="https://slideslive.com/38931314/profiling-immunoglobulin-repertoires-across-multiple-human-tissues-using-rnaseq" target="_blank">slideslive</a>] <br>

&emsp; Shibiao Wan: Hyper-fast and accurate clustering of ultra-large-scale single-cell data with ensemble random projection
      [<a href="2020/papers/WCBICML2020_paper_22.pdf" target="_blank">paper&nbsp;22</a>]
      [<a href="https://slideslive.com/38931315/sharp-singlecell-rnaseq-hyperfast-and-accurate-clustering-via-ensemble-random-projection" target="_blank">slideslive</a>] <br>

&emsp;	William Hsu: Discovering and interpreting transcriptomic drivers of imaging traits using neural networks
      [<a href="2020/papers/WCBICML2020_paper_57.pdf" target="_blank">paper&nbsp;57</a>]
      [<a href="https://slideslive.com/38931316/discovering-and-interpreting-transcriptomic-drivers-of-imaging-traits-using-neural-networks" target="_blank">slideslive</a>] <br>

&emsp;	Jacob Schreiber: Learning a latent representation of human genomics using Avocado
      [<a href="2020/papers/WCBICML2020_paper_6.pdf" target="_blank">paper&nbsp;6</a>]
      [<a href="https://slideslive.com/38931675/learning-a-latent-representation-of-human-genomics-using-avocado" target="_blank">slideslive</a>] <br> <br>


      11:55 am - 1pm &emsp; Poster session 1<br>
      &emsp;Variational autoencoders with flexible priors enable robust distribution learning on single-cell RNA sequencing data : Leander Dony
      [<a href="2020/papers/WCBICML2020_paper_37.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_37.pdf" target="_blank">poster</a>] <br>
      
&emsp;scGNN: scRNA-seq Dropout Imputation via Induced Hierarchical Cell Similarity Graph: Kexin Huang 
      [<a href="2020/papers/WCBICML2020_paper_17.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_17.pdf" target="_blank">poster</a>] <br>
      
&emsp;Gene Expression Imputation with Generative Adversarial Imputation Nets: Ramon Viñas Torné 
      [<a href="2020/papers/WCBICML2020_paper_35.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_35.pdf" target="_blank">poster</a>] <br>
      
&emsp;Out-of-distribution prediction with disentangled representations for single-cell RNA sequencing data: Mohammad Lotfollahi 
      [<a href="2020/papers/WCBICML2020_paper_41.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_41.pdf" target="_blank">poster</a>] <br>
      
&emsp;Cross Attentive Antibody-Antigen Interaction Prediction with Multi-task Learning: Kyohei Koyama 
      [<a href="2020/papers/WCBICML2020_paper_5.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_5.pdf" target="_blank">poster</a>] <br>
      
&emsp;DNA folding features prediction with Recurrent Neural Networks using epigenetic data: Michal Rozenwald 
      [<a href="2020/papers/WCBICML2020_paper_7.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_7.pdf" target="_blank">poster</a>] <br>
      
&emsp;Deriving Cell Type-Specific Directed Weighted Signed Regulatory Networks from Single-Cell RNA Sequencing Data: Larisa Morales 
      [<a href="2020/papers/WCBICML2020_paper_28.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_28.pdf" target="_blank">poster</a>] <br>
      
&emsp;Learning Heat Diffusion for Network Alignment: Sisi Qu 
      [<a href="2020/papers/WCBICML2020_paper_68.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_68.pdf" target="_blank">poster</a>] <br>
      
&emsp;Identifying cross-tissue signaling between genes from biomedical literature: Aditya Jadhav 
      [<a href="2020/posters/WCBICML2020_poster_69.pdf" target="_blank">poster</a>] <br>
      
&emsp;PaccMann<sup>RL</sup> on SARS-CoV-2: Designing antiviral candidates with conditional generative models: Nil Adell Mill 
      [<a href="2020/papers/WCBICML2020_paper_53.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_53.pdf" target="_blank">poster</a>] <br><br>
       
      1-2pm &emsp; Poster session 2<br> 
        &emsp;Learning the locomotion behaviour of lizards transfers across environments: Kathleen Foster 
      [<a href="2020/papers/WCBICML2020_paper_2.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_2.pdf" target="_blank">poster</a>] <br>
      
&emsp;Optimal Transport using GANs for Lineage Tracing: Neha Prasad 
      [<a href="2020/papers/WCBICML2020_paper_8.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_8.pdf" target="_blank">poster</a>] <br>
      
&emsp;Systematic characterization of generative models for de novo design of regulatory DNA: Nic Fishman
      [<a href="2020/papers/WCBICML2020_paper_46.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_46.pdf" target="_blank">poster</a>] <br>
      
&emsp;GPU-Accelerated SVM Learning for Extremely Fast Massive-Scale Proteomics Classification: John Halloran 
      [<a href="2020/papers/WCBICML2020_paper_18.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_18.pdf" target="_blank">poster</a>] <br>
      
      
&emsp;MTSplice predicts effects of genetic variants on tissue-specific splicing: Jun Cheng 
      [<a href="2020/papers/WCBICML2020_paper_47.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_47.pdf" target="_blank">poster</a>] <br>
      
&emsp;Auto-encoders with fibered latent spaces: A geometric approach to batch correction: Tariq Daouda 
      [<a href="2020/papers/WCBICML2020_paper_63.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_63.pdf" target="_blank">poster</a>] <br>
      
&emsp;Topological Methods for fMRI Data: Bastian Rieck
      [<a href="2020/papers/WCBICML2020_paper_9.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_9.pdf" target="_blank">poster</a>] <br>
      
&emsp;Identification of Epitope-TCR Binding Using A Generative Adversarial Network Model: Seojin Bang 
      [<a href="2020/papers/WCBICML2020_paper_13.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_13.pdf" target="_blank">poster</a>] <br>
      
&emsp;Learning Cancer Progression Network from Mutation Allele Frequencies: Amir Asiaee 
      [<a href="2020/papers/WCBICML2020_paper_36.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_36.pdf" target="_blank">poster</a>] <br>
      
&emsp;TENET: Gene network reconstruction using single cell transcriptomic data reveals key factors for embryonic stem cell differentiation: Junil Kim 
      [<a href="2020/papers/WCBICML2020_paper_48.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_48.pdf" target="_blank">poster</a>] <br>
      
&emsp;Determining causal interactions learned by genomic DL models with in silico mutagenesis and Mendelian randomization: Stephen Malina 
      [<a href="2020/posters/WCBICML2020_poster_34.pdf" target="_blank">poster</a>] <br>
      <br>
      
        <h6 class="heading-6">Session 3 (Chair: Jill Gallaher)</h6>

      2-2:30 pm &emsp; Invited Talk 3: <b> Thomas Fuchs: </b> Clinical-grade Artificial Intelligence: Hype and Hope for Cancer Care [Live only] <br> <br>

      2:30 - 2:50 pm &emsp; Contributed Talk 3: Guillaume Jaume: Towards Explainable Graph Representations in Digital Pathology
      [<a href="2020/papers/WCBICML2020_paper_11.pdf" target="_blank">paper&nbsp;11</a>]
      [<a href="https://slideslive.com/38930613/towards-explainable-graph-representations-in-digital-pathology" target="_blank">slideslive</a>] <br> <br>

      2:50 - 3:15 pm  &emsp; Spotlights:<br>
&emsp;	Kathleen Lois Foster: Learning the locomotion behaviour of lizards transfers across environments
      [<a href="2020/papers/WCBICML2020_paper_2.pdf" target="_blank">paper&nbsp;2</a>]
      [<a href="https://slideslive.com/38930615/learning-the-locomotion-behaviour-of-lizards-transfers-across-environments" target="_blank">slideslive</a>] <br>

&emsp;	Jacob Schreiber: Ledidi: Designing genome edits that induce functional activity
      [<a href="2020/papers/WCBICML2020_paper_23.pdf" target="_blank">paper&nbsp;23</a>]
      [<a href="https://slideslive.com/38930903/ledidi-designing-genomic-edits-that-induce-functional-activity" target="_blank">slideslive</a>] <br>

&emsp; 46	Nic Fishman: Systematic characterization of generative models for de novo design of regulatory DNA 
      [<a href="2020/papers/WCBICML2020_paper_46.pdf" target="_blank">paper&nbsp;46</a>]
      [<a href="https://slideslive.com/38930904/systematic-characterization-of-generative-models-for-de-novo-design-of-regulatory-dna" target="_blank">slideslive</a>] <br>

&emsp;	Alex Karollus: Predicting Mean Ribosome Load for 5’UTR of any length using Deep Learning
      [<a href="https://slideslive.com/38930905/predicting-mean-ribosome-load-for-5utr-of-any-length-using-deep-learning" target="_blank">slideslive</a>] <br>

&emsp;	Pinar Demetci: Gromov–Wasserstein Optimal Transport to Align Single-Cell Multi-Omics Data 
      [<a href="2020/papers/WCBICML2020_paper_51.pdf" target="_blank">paper&nbsp;51</a>]
      [<a href="https://slideslive.com/38930906/gromovwasserstein-optimal-transport-to-align-singlecell-multiomics-data" target="_blank">slideslive</a>] <br> <br>


      3:15 - 4:15 pm &emsp;  Poster session 3<br> 
      &emsp;	Normalisr: inferring single-cell differential and co-expression with linear association testing: Lingfei Wang 
      [<a href="2020/posters/WCBICML2020_poster_16.pdf" target="_blank">poster</a>] <br>
      
&emsp;	Gromov–Wasserstein Optimal Transport to Align Single-Cell Multi-Omics Data: Rebecca Santorella 
      [<a href="2020/papers/WCBICML2020_paper_51.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_51.pdf" target="_blank">poster</a>] <br>
      
&emsp;	Ledidi: Designing genome edits that induce functional activity Jacob Schreiber 
      [<a href="2020/papers/WCBICML2020_paper_23.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_23.pdf" target="_blank">poster</a>] <br>
      
&emsp;	Predicting Mean Ribosome Load for 5’UTR of any length using Deep Learning: Alexander Karollus 
      [<a href="2020/posters/WCBICML2020_poster_49.pdf" target="_blank">poster</a>] <br>
      
&emsp;	Look at the Loss: Towards Robust Detection of False Positive Feature Interactions Learned by Neural Networks on Genomic Data: Avanti Srikumar 
      [<a href="2020/papers/WCBICML2020_paper_58.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_58.pdf" target="_blank">poster</a>] <br>
      
&emsp;	Biologically-relevant transfer learning improves transcription factor binding prediction: Manu Saraswat 
      [<a href="2020/papers/WCBICML2020_paper_66.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_66.pdf" target="_blank">poster</a>] <br>
      
&emsp;	Slide-free MUSE Microscopy to H&E Histology Modality Conversion via Unpaired Image-to-Image Translation GAN Models: Tanishq Abraham 
      [<a href="2020/papers/WCBICML2020_paper_67.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_67.pdf" target="_blank">poster</a>] <br>
      
&emsp;	DeepPurpose: a Deep Learning Based Drug Repurposing Toolkit: Kexin Huang 
      [<a href="2020/papers/WCBICML2020_paper_15.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_15.pdf" target="_blank">poster</a>] <br>
      
&emsp;	Using deep learning on chest CT to track COVID-19 patients: Edward Lee 
      [<a href="2020/posters/WCBICML2020_poster_26.pdf" target="_blank">poster</a>] <br>
      
&emsp;	ChronoStrain: Sequence quality and time aware strain tracking with shotgun metagenomic data: YounHun Kim 
      [<a href="2020/posters/WCBICML2020_poster_31.pdf" target="_blank">poster</a>] <br>
      
&emsp;	Supervised Tumor Cell Subtype Identification via SCAN: Russell Kunes 
      [<a href="2020/papers/WCBICML2020_paper_38.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_38.pdf" target="_blank">poster</a>] <br>
      
&emsp;  Data-driven Variable-length Segmentation of Biological Sequences: Applications in Proteomics and Metagenomics: Ehsan Asgari 
      [<a href="2020/papers/WCBICML2020_paper_60.pdf" target="_blank">paper</a>] 
      [<a href="2020/posters/WCBICML2020_poster_60.pdf" target="_blank">poster</a>] <br>
      <br>
        
        <h6 class="heading-6">Session 4 (Chair: Yubin Xie)</h6>
        
      4:15 - 4:45 pm &emsp; Invited Talk 4: <b> Debora Marks: </b> Learning and designing biology with neural machines [Live only] <br> <br>
        
      4:45 - 5:05 pm &emsp; Contributed Talk 4: Minxing Pang: Representation Learning and Translation between the Mouse and Human Brain using a Deep Transformer Architecture 
      [<a href="2020/papers/WCBICML2020_paper_29.pdf" target="_blank">paper&nbsp;29</a>]
      [Live only]<br> <br>
        
      5:05-5:30 pm &emsp; Award ceremony and closing remarks<br>
        
      </p>
    </div>
  </section>
  
  <header id="speaker" class="hero">
    <div class="w-container">
      <h4 class="heading-3">Invited speakers</h4>
      <p class="paragraph-10">*Listed alphabetically</p>
    </div>
    <div class="flex-container w-container"><img src="images/thomas-fuchs-3x2.jpg" width="225" alt="" class="image-3">
      <div class="hero-image-mask"></div>
      <div class="div-block-4">
        <h5 class="heading-9"><a href="http://thomasfuchslab.org" target="_blank">Thomas Fuchs</a></h5>
        <p class="paragraph-7">Group leader, Memorial Sloan Kettering Cancer Center<br>Associate Professor, Weill Cornell Medicine</p>
      </div>
    </div>
    <div class="flex-container w-container"><img src="images/unnamed.jpg" width="225" alt="" class="image-2">
      <div class="hero-image-mask"></div>
      <div class="div-block-5">
        <h5 class="heading-10"><a href="https://www.deboramarkslab.com/" target="_blank">Debora Marks</a></h5>
        <p class="paragraph-8">Associate Professor at Harvard Medical School <br>Associate member at the Broad Institute<br> </p>
      </div>
    </div>
    <div class="flex-container w-container"><img src="images/YBZZMQ5e.jpg" width="225" alt="" class="image">
      <div class="hero-image-mask"></div>
      <div class="div-block-6">
        <h5 class="heading-11"><a href="https://www.helmholtz-muenchen.de/icb/research/groups/machine-learning/overview/index.html" target="_blank">Fabian Theis</a></h5>
        <p class="paragraph-9">Group Leader, Helmholtz Zentrum München <br>Head of Institute of Computational Biology</p>
      </div>
    </div>
    <div class="flex-container w-container"><img src="images/ogt_1.jpg" width="225" alt="" class="image">
      <div class="hero-image-mask"></div>
      <div class="div-block-6">
        <h5 class="heading-8"><a href="https://function.princeton.edu" target="_blank" class="link-5">Olga Troyanskaya</a></h5>
        <p class="paragraph-9">Professor, Deputy Director of Genomics,<br>Princeton University<br><br></p>
      </div>
    </div>
  </header>
  
  
  <header id="papers" class="hero">
    <div class="w-container">
      <h4 class="heading-3">Accepted Submissions</h4>
      <br>
      <h5 class="heading-4">Contributed Talks</h5>
      <p class="paragraph-10" align="justify">

        Paper 20:	Geoff Fudenberg, David R Kelley and Katherine S Pollard. <i> Predicting 3D genome folding from DNA sequence</i> 
        [<a href="2020/papers/WCBICML2020_paper_20.pdf" target="_blank">paper</a>] 
        [<a href="https://slideslive.com/38930902/predicting-3d-genome-folding-from-dna-sequence" target="_blank">slideslive</a>] <br>

        Paper 11:	Guillaume Jaume, Pushpak Pati, Antonio Foncubierta Rodriguez, Jean-Philippe Thiran, Orcun Goksel and Maria Gabrani. <i> Towards Explainable Graph Representations in Digital Pathology</i> 
        [<a href="2020/papers/WCBICML2020_paper_11.pdf" target="_blank">paper</a>] 
        [<a href="https://slideslive.com/38930613/towards-explainable-graph-representations-in-digital-pathology" target="_blank">slideslive</a>] <br>

        Paper 29:	Minxing Pang and Jesper Tegnér. <i> Representation Learning and Translation between the Mouse and Human Brain using a Deep Transformer Architecture</i> 
        [<a href="2020/papers/WCBICML2020_paper_29.pdf" target="_blank">paper</a>] [Live only]<br>

        Paper 44:	Jacob C. Kimmel and David R. Kelley. <i> scNym: Semi-supervised adversarial neural networks for single cell classification</i>
        [<a href="2020/papers/WCBICML2020_paper_44.pdf" target="_blank">paper</a>] 
        [<a href="https://slideslive.com/38930608/scnym-semisupervised-adversarial-neural-networks-for-single-cell-classification" target="_blank">slideslive</a>] <br>

      </p>
    <br>
      <h5 class="heading-4">Spotlights & Posters</h5>
      <p class="paragraph-10" align="justify">

        Paper 8: Neha Prasad, Karren Yang and Caroline Uhler. <i> Optimal Transport using GANs for Lineage Tracing</i> 
        [<a href="2020/papers/WCBICML2020_paper_8.pdf" target="_blank">paper</a>] 
        [<a href="2020/posters/WCBICML2020_poster_8.pdf" target="_blank">poster</a>] 
        [<a href="https://slideslive.com/38931306/superot-optimal-transport-methodology-for-biological-lineage-tracing" target="_blank">slideslive</a>] <br>

        Paper 16: Lingfei Wang, Jacques Deguine and Ramnik Xavier. <i> Normalisr: inferring single-cell differential and co-expression with linear association testing </i>  
        [<a href="2020/posters/WCBICML2020_poster_16.pdf" target="_blank">poster</a>] 
        [<a href="https://slideslive.com/38931307/normalisr-inferring-singlecell-differential-and-coexpression-with-linear-association-testing" target="_blank">slideslive</a>] <br>

        Paper 17: Kexin Huang. <i> scGNN: scRNA-seq Dropout Imputation via Induced Hierarchical Cell Similarity Graph</i> 
        [<a href="2020/papers/WCBICML2020_paper_17.pdf" target="_blank">paper</a>] 
        [<a href="2020/posters/WCBICML2020_poster_17.pdf" target="_blank">poster</a>] 
        [<a href="https://slideslive.com/38931308/scgnn-scrnaseq-dropout-imputation-via-induced-hierarchical-cell-similarity-graph" target="_blank">slideslive</a>] <br>

        Paper 37: Leander Dony, Martin Koenig, David S. Fischer and Fabian J. Theis. <i> Variational autoencoders with flexible priors enable robust distribution learning on single-cell RNA sequencing data</i> 
        [<a href="2020/papers/WCBICML2020_paper_37.pdf" target="_blank">paper</a>] 
        [<a href="2020/posters/WCBICML2020_poster_37.pdf" target="_blank">poster</a>] 
        [<a href="https://slideslive.com/38931309/variational-autoencoders-with-flexible-priors-enable-robust-distribution-learning-on-singlecell-rna-sequencing-data" target="_blank">slideslive</a>] <br>

        Paper 41: Mohammad Lotfollahi, Leander Dony, Harshita Agarwala and Fabian Theis. <i> Out-of-distribution prediction with disentangled representations for single-cell RNA sequencing data</i> 
        [<a href="2020/papers/WCBICML2020_paper_41.pdf" target="_blank">paper</a>] 
        [<a href="2020/posters/WCBICML2020_poster_41.pdf" target="_blank">poster</a>] 
        [<a href="https://slideslive.com/38931310/outofdistribution-prediction-with-disentangled-representations-for-scrnaseq-data" target="_blank">slideslive</a>] 
<br>

        Paper 2: Kathleen Lois Foster and Alessandro Maria Selvitella. <i> Learning the locomotion behaviour of lizards transfers across environments</i> 
        [<a href="2020/papers/WCBICML2020_paper_2.pdf" target="_blank">paper</a>] 
        [<a href="2020/posters/WCBICML2020_poster_2.pdf" target="_blank">poster</a>] 
        [<a href="https://slideslive.com/38930615/learning-the-locomotion-behaviour-of-lizards-transfers-across-environments" target="_blank">slideslive</a>] 
<br>

        Paper 23: Jacob Schreiber, Yang Young Lu and William Stafford Noble. <i> Ledidi: Designing genome edits that induce functional activity</i>  
        [<a href="2020/papers/WCBICML2020_paper_23.pdf" target="_blank">paper</a>] 
        [<a href="2020/posters/WCBICML2020_poster_23.pdf" target="_blank">poster</a>]
        [<a href="https://slideslive.com/38930903/ledidi-designing-genomic-edits-that-induce-functional-activity" target="_blank">slideslive</a>] <br>

        Paper 46: Nic Fishman, Avanti Shrikumar, Georgi Marinov and Anshul Kundaje. <i> Systematic characterization of generative models for de novo design of regulatory DNA</i> 
        [<a href="2020/papers/WCBICML2020_paper_46.pdf" target="_blank">paper</a>]
        [<a href="2020/posters/WCBICML2020_poster_46.pdf" target="_blank">poster</a>]
        [<a href="https://slideslive.com/38930904/systematic-characterization-of-generative-models-for-de-novo-design-of-regulatory-dna" target="_blank">slideslive</a>] <br>

        Paper 49: Alexander Karollus, Ziga Avsec and Julien Gagneur. <i> Predicting Mean Ribosome Load for 5’UTR of any length using Deep Learning</i> 
        [<a href="2020/posters/WCBICML2020_poster_49.pdf" target="_blank">poster</a>]
        [<a href="https://slideslive.com/38930905/predicting-mean-ribosome-load-for-5utr-of-any-length-using-deep-learning" target="_blank">slideslive</a>] <br>

        Paper 51: Pinar Demetci, Rebecca Santorella, Bjorn Sandstede, William Stafford Noble and Ritambhara Singh. <i> Gromov–Wasserstein Optimal Transport to Align Single-Cell Multi-Omics Data</i>
        [<a href="2020/papers/WCBICML2020_paper_51.pdf" target="_blank">paper</a>]
        [<a href="2020/posters/WCBICML2020_poster_51.pdf" target="_blank">poster</a>]
        [<a href="https://slideslive.com/38930906/gromovwasserstein-optimal-transport-to-align-singlecell-multiomics-data" target="_blank">slideslive</a>] <br>

      </p>
     <br>
      
      <h5 class="heading-4">Posters</h5>
      <p class="paragraph-10" align="justify">
      Paper 35: Ramon Viñas, Tiago Azevedo, Eric Gamazon and Pietro Liò. <i> Gene Expression Imputation with Generative Adversarial Imputation Nets</i>
        [<a href="2020/papers/WCBICML2020_paper_35.pdf" target="_blank">paper</a>]
        [<a href="2020/posters/WCBICML2020_poster_35.pdf" target="_blank">poster</a>] <br>

      Paper 5: Kyohei Koyama, Kotaro Kamiya and Shion Honda. <i> Cross Attentive Antibody-Antigen Interaction Prediction with Multi-task Learning</i>
        [<a href="2020/papers/WCBICML2020_paper_5.pdf" target="_blank">paper</a>]
        [<a href="2020/posters/WCBICML2020_poster_5.pdf" target="_blank">poster</a>] <br>

      Paper 7: Michal Rozenwald, Aleksandra Galitsyna, Mikhail S. Gelfand and Ekaterina Khrameeva. <i> DNA folding features prediction with Recurrent Neural Networks using epigenetic data</i> 
        [<a href="2020/papers/WCBICML2020_paper_7.pdf" target="_blank">paper</a>]
        [<a href="2020/posters/WCBICML2020_poster_7.pdf" target="_blank">poster</a>] <br>

      Paper 18: John Halloran and David Rocke. <i> GPU-Accelerated SVM Learning for Extremely Fast Massive-Scale Proteomics Classification</i>
        [<a href="2020/papers/WCBICML2020_paper_18.pdf" target="_blank">paper</a>]
        [<a href="2020/posters/WCBICML2020_poster_18.pdf" target="_blank">poster</a>] <br>
 
      Paper 47: Jun Cheng, Muhammed Hasan Çelik, Anshul Kundaje and Julien Gagneur. <i> MTSplice predicts effects of genetic variants on tissue-specific splicing</i>
        [<a href="2020/papers/WCBICML2020_paper_47.pdf" target="_blank">paper</a>]
        [<a href="2020/posters/WCBICML2020_poster_47.pdf" target="_blank">poster</a>] <br>

      Paper 53: Jannis Born, Matteo Manica, Joris Cadow, Greta Markert, Nil Adell Mill, Modestas Filipavicius and Maria Rodriguez Martinez. <i> PaccMann<sup>RL</sup> on SARS-CoV-2: Designing antiviral candidates with conditional generative models</i>
        [<a href="2020/papers/WCBICML2020_paper_53.pdf" target="_blank">paper</a>]
        [<a href="2020/posters/WCBICML2020_poster_53.pdf" target="_blank">poster</a>]<br>

      Paper 58: Mara Finkelstein, Avanti Shrikumar and Anshul Kundaje. <i> Look at the Loss: Towards Robust Detection of False Positive Feature Interactions Learned by Neural Networks on Genomic Data</i>
        [<a href="2020/papers/WCBICML2020_paper_58.pdf" target="_blank">paper</a>]
        [<a href="2020/posters/WCBICML2020_poster_58.pdf" target="_blank">poster</a>]<br>

      Paper 66: Gherman Novakovsky, Manu Saraswat, Oriol Fornes and Wyeth Wasserman. <i> Biologically-relevant transfer learning improves transcription factor binding prediction</i>
        [<a href="2020/papers/WCBICML2020_paper_66.pdf" target="_blank">paper</a>]
        [<a href="2020/posters/WCBICML2020_poster_66.pdf" target="_blank">poster</a>]<br>

      Paper 67: Tanishq Abraham, Andrew Shaw, Daniel O'Connor, Austin Todd and Richard Levenson. <i> Slide-free MUSE Microscopy to H&E Histology Modality Conversionvia Unpaired Image-to-Image Translation GAN Models</i>
        [<a href="2020/papers/WCBICML2020_paper_67.pdf" target="_blank">paper</a>]
        [<a href="2020/posters/WCBICML2020_poster_67.pdf" target="_blank">poster</a>]<br>

      Paper 63: Tariq Daouda, Reda Chhaibi, Prudencio Tossou and Alexandra-Chloé Villani. <i> Auto-encoders with fibered latent spaces: A geometric approach to batch correction</i>
        [<a href="2020/papers/WCBICML2020_paper_63.pdf" target="_blank">paper</a>]
        [<a href="2020/posters/WCBICML2020_poster_63.pdf" target="_blank">poster</a>]<br>

      Paper 9: Bastian Rieck, Tristan Yates, Guy Wolf, Nicholas Turk-Browne and Smita Krishnaswamy. <i> Topological Methods for fMRI Data</i>
        [<a href="2020/papers/WCBICML2020_paper_9.pdf" target="_blank">paper</a>]
        [<a href="2020/posters/WCBICML2020_poster_9.pdf" target="_blank">poster</a>]<br>

      Paper 13: Seojin Bang and Heewook Lee. <i> Identification of Epitope-TCR Binding Using A Generative Adversarial Network Model</i>
        [<a href="2020/papers/WCBICML2020_paper_13.pdf" target="_blank">paper</a>] 
        [<a href="2020/posters/WCBICML2020_poster_13.pdf" target="_blank">poster</a>]<br>

      Paper 15: Kexin Huang, Tianfan Fu, Cao Xiao, Lucas Glass and Jimeng Sun. <i> DeepPurpose: a Deep Learning Based Drug Repurposing Toolkit</i>
        [<a href="2020/papers/WCBICML2020_paper_15.pdf" target="_blank">paper</a>]
        [<a href="2020/posters/WCBICML2020_poster_15.pdf" target="_blank">poster</a>]<br>

      Paper 26: Edward Lee, Jiangdian Song, Hongmei Wang, Wei Zhang, Jimmy Zheng, Michelle Han, Jayne Seekins, Simon Wong, Kexue Deng and Kristen Yeom. <i> Using deep learning on chest CT to track COVID-19 patients</i>
        [<a href="2020/posters/WCBICML2020_poster_26.pdf" target="_blank">poster</a>]<br>

      Paper 28: Larisa Morales-Soto, Juan P. Bernal-Tamayo, Robert Lehman, Balsamy Subash, Xabier Martinez-de-Morentin, Amaia Vilas-Zornoza, Patxi San-Martin, David Lara, Felipe Prosper, David Gomez-Cabrero, Narsis Kiani and Jesper Tegner. <i> Deriving Cell Type-Specific Directed Weighted Signed Regulatory Networks from Single-Cell RNA Sequencing Data</i>
        [<a href="2020/papers/WCBICML2020_paper_28.pdf" target="_blank">paper</a>]
        [<a href="2020/posters/WCBICML2020_poster_28.pdf" target="_blank">poster</a>]<br>

      Paper 31: Younhun Kim, Sawal Acharya, Daniel Alfonsetti, Georg Gerber, Bonnie Berger and Travis Gibson. <i> ChronoStrain: Sequence quality and time aware strain tracking with shotgun metagenomic data</i>
        [<a href="2020/posters/WCBICML2020_poster_31.pdf" target="_blank">poster</a>]<br>

      Paper 36: Mohammad Sadegh Akhondzadeh, Alireza Omidi, Zeinab Maleki, Kevin R. Coombes, Amanda E. Toland and Amir Asiaee. <i> Learning Cancer Progression Network from Mutation Allele Frequencies</i>
        [<a href="2020/papers/WCBICML2020_paper_36.pdf" target="_blank">paper</a>]
        [<a href="2020/posters/WCBICML2020_poster_36.pdf" target="_blank">poster</a>]<br>

      Paper 34: Stephen Malina, Daniel Cizin and David Knowles. <i>	Determining causal interactions learned by genomic DL models with in silico mutagenesis and Mendelian randomization</i>
        [<a href="2020/posters/WCBICML2020_poster_34.pdf" target="_blank">poster</a>]<br>

      Paper 38: Russell Kunes, Siyu He, Yang Xiao, Simon Tavare and David Knowles. <i>	Supervised Tumor Cell Subtype Identification via SCAN</i>
        [<a href="2020/papers/WCBICML2020_paper_38.pdf" target="_blank">paper</a>]
        [<a href="2020/posters/WCBICML2020_poster_38.pdf" target="_blank">poster</a>]<br>

      Paper 48: Junil Kim, Simon Toftholm Jakobsen, Kedar Nath Natarajan and Kyoung Jae Won. <i> TENET: Gene network reconstruction using single cell transcriptomic data \\ reveals key factors for embryonic stem cell differentiation</i>
        [<a href="2020/papers/WCBICML2020_paper_48.pdf" target="_blank">paper</a>]
        [<a href="2020/posters/WCBICML2020_poster_48.pdf" target="_blank">poster</a>]<br>

      Paper 60: Ehsaneddin Asgari, Philipp Muench, Till-Robin Lesker, Alice C. Mchardy and Mohammad R. K. Mofrad. <i> Data-driven Variable-length Segmentation of Biological Sequences: Applications in Proteomics and Metagenomics</i>
        [<a href="2020/papers/WCBICML2020_paper_60.pdf" target="_blank">paper</a>] 
        [<a href="2020/posters/WCBICML2020_poster_60.pdf" target="_blank">poster</a>]<br>

      Paper 68: Sisi Qu, Mengmeng Xu, Bernard Ghanem and Jesper Tegner. <i> Learning Heat Diffusion for Network Alignment</i>
        [<a href="2020/papers/WCBICML2020_paper_68.pdf" target="_blank">paper</a>]
        [<a href="2020/posters/WCBICML2020_poster_68.pdf" target="_blank">poster</a>]<br>

      Paper 69: Aditya Jadhav and Manikandan Narayanan. <i> Identifying cross-tissue signaling between genes from biomedical literature</i> 
        [<a href="2020/posters/WCBICML2020_poster_69.pdf" target="_blank">poster</a>]<br>

      </p>
     <br>
      <h5 class="heading-4">Highlights of Published Work</h5>
      <p class="paragraph-10" align="justify">

      Paper 3: Sanjiv Dwivedi, Andreas Tjärnberg, Jesper Tegnér and Mika Gustafsson. <i> Deriving Disease Modules from the Compressed Transcriptional Space Embedded in a Deep Autoencoder</i>
        [<a href="2020/papers/WCBICML2020_paper_3.pdf" target="_blank">paper</a>]
        [<a href="https://slideslive.com/38931311/deriving-disease-modules-from-the-compressed-transcriptional-space-embedded-in-a-deep-autoencoder" target="_blank">slideslive</a>] <br>

      Paper 4: Iman Deznabi, Büşra Arabacı, Mehmet Koyuturk and Oznur Tastan. <i> DeepKinZero: Zero-Shot Learning for Predicting Kinase-Phosphosite Associations</i>
        [<a href="2020/papers/WCBICML2020_paper_4.pdf" target="_blank">paper</a>]
        [<a href="https://slideslive.com/38931312/deepkinzero-zeroshot-learning-for-predicting-kinasephosphosite-associations-involving-understudied-kinases" target="_blank">slideslive</a>] <br>

      Paper 14: Christian Matek, Simone Schwarz, Karsten Spiekermann and Carsten Marr. <i> A publicly available database for developing machine learning applications to differentiate leukocytes and recognise malignant cells in peripheral blood</i>
        [<a href="2020/papers/WCBICML2020_paper_14.pdf" target="_blank">paper</a>]
        [<a href="https://slideslive.com/38931313/a-publicly-available-dataset-for-developing-machine-learning-applications-to-differentiate-leukocytes-and-recognise-malignant-cells-in-peripheral-blood" target="_blank">slideslive</a>] <br>

      Paper 21: Serghei Mangul, Igor Mandric and Jeremy Rotman. <i> Profiling immunoglobulin repertoires across multiple human tissues using RNA Sequencing</i>  
        [<a href="2020/papers/WCBICML2020_paper_21.pdf" target="_blank">paper</a>] 
        [<a href="https://slideslive.com/38931314/profiling-immunoglobulin-repertoires-across-multiple-human-tissues-using-rnaseq" target="_blank">slideslive</a>] <br>

      Paper 22: Shibiao Wan, Junil Kim and Kyoung Jae Won. <i> Hyper-fast and accurate clustering of ultra-large-scale single-cell data with ensemble random projection</i>
        [<a href="2020/papers/WCBICML2020_paper_22.pdf" target="_blank">paper</a>]
        [<a href="https://slideslive.com/38931315/sharp-singlecell-rnaseq-hyperfast-and-accurate-clustering-via-ensemble-random-projection" target="_blank">slideslive</a>]<br>

      Paper 57: Nova Smedley, Suzie El-Saden and William Hsu. <i> Discovering and interpreting transcriptomic drivers of imaging traits using neural networks</i>  
        [<a href="2020/papers/WCBICML2020_paper_57.pdf" target="_blank">paper</a>]
        [<a href="https://slideslive.com/38931316/discovering-and-interpreting-transcriptomic-drivers-of-imaging-traits-using-neural-networks" target="_blank">slideslive</a>] <br>

      Paper 6: Jacob Schreiber and William Stafford Noble. <i> Learning a latent representation of human genomics using Avocado</i>
        [<a href="2020/papers/WCBICML2020_paper_6.pdf" target="_blank">paper</a>] 
        [<a href="https://slideslive.com/38931675/learning-a-latent-representation-of-human-genomics-using-avocado" target="_blank">slideslive</a>] <br>
      </p>
      
    </div>
  </header>
    
  
  <main id="OC" class="main-content">
    <div class="container-2 w-container">
      <h4 class="heading-4">Organizing committee</h4>
    </div>
    <div class="w-container">
      <p class="paragraph-2"><span>*Names are listed alphabetically<br>
        </span>Delasa Aghamirzaie, Illumina<br>
        Alexander Anderson, Moffitt Cancer Center<br>
        Elham Azizi, Columbia University<br>
        Abdoulaye Baniré Diallo, Université du Québec à Montréal<br>
        Cassandra Burdziak, Memorial Sloan Kettering Cancer Center<br>
        Wajdi Dhifli, University of Lille<br>
        Jill Gallaher, Moffitt Cancer Center<br>
        Anshul Kundaje, Stanford University<br>
        Engelbert Mephu Nguifo, Blaise Pascal University<br>
        Dana Pe&#x27;er, Memorial Sloan Kettering Cancer Center (MSKCC)<br>
        Sandhya Prabhakaran, Moffitt Cancer Center<br>
        Amine Remita, Université du Québec à Montréal<br>
        Mark Robertson-Tessi, Moffitt Cancer Center<br>
        Wesley Tansey, Columbia University<br>Julia E. Vogt, University of Basel<br>
        Yubin Xie, Memorial Sloan Kettering Cancer Center</p>
    </div>
  </main>
  <main id="main-content" class="main-content-2">
    <div class="w-container">
      <div class="rich-text-block-4 w-richtext">
        <h4>Program Committee</h4>
        <p class="paragraph-3">Delasa Aghamirzaie, Illumina <br>
Mohammed Alquraishi, Harvard University <br>         
Sabeur Aridhi, University of Lorraine <br>
Elham Azizi, Columbia University <br>
Alexis Battle, Johns Hopkins University <br>          
Cassandra Burdziak, MSKCC <br> 
Mathieu Carrière, Columbia University <br>
Ibrahim Chamseddine, Harvard Medical School - MGH <br>
Ahmet Coskun, Georgia Insitute of Technology <br>
Elisabetta De Maria, University of Nice Sophia Antipolis<br>
Marcilio De Souto, Federal University of Rio Grande do Norte<br>
Wajdi Dhifli, University of Lille <br>
Mohamed Elati, University of Lille <br>
Jason Ernst, UCLA <br>
Mervin Fansler, MSKCC <br>
Meghan Ferrall-Fairbanks, Moffitt Cancer Center <br>	
Jill Gallaher, Moffitt Cancer Center <br>
Christopher Garay, MITRE <br>
Casey Greene, University of Pennsylvania <br>
Mengting Gu, Yale University <br>
Niina Haiminen, IBM Research <br>
Adrian Heilbut, Kallyope <br>
David Knowles, NYGC & Columbia University <br>
Anshul Kundaje, Stanford University <br>
Sheng Liu, Indiana University School of Medicine <br>
Romain Lopez, UC Berkeley <br>
Yuheng Lu, Harvard University
Engelbert Mephu Nguifo, Blaise Pascal University <br>
Quaid Morris, MSKCC <br>
Sarvesh Nikumbh, Max-Planck-Institut für Informatik <br>
Mor Nitzan, Harvard University <br>
Chetanya Pandya, Bluebird bio<br> 
Sandhya Prabhakaran, Moffitt Cancer Center<br> 
Amine Remita, Université du Québec à Montréal <br>
Luis Rueda, University of Windsor <br>
Andrew Schaumberg, MSKCC <br>
Manu Setty, MSKCC <br>
Yuelin Shi, Caltech <br>
Anne Siegel, University of Rennes <br>
Ayshwarya Subramanian, Broad Institute <br>
Wesley Tansey, Columbia University & MSKCC <br>
Filippo Utro, IBM <br>
Yubin Xie, MSKCC <br>
Chensu Xie, Weill Cornell <br>	
Bo Yuan, Harvard University <br>          
Han Yuan, MSKCC <br>
Marinka Zitnik, Harvard University  </p>
      </div>
    </div>
  </main>
  <section id="sponsor" class="cards-section">
    <div class="w-container">
      <h4 class="heading-4">Sponsors</h4>
    </div>
    <div class="centered-container w-container">

      <div class="cards-grid-container">

        
        <div id="w-node-0d05fd79f6e4-1853825c" class="div-block-2">
          <div class="cards-image-mask"><img src="images/Unknown-1.png" alt="" class="cards-image"></div>
          <h6 class="heading-7"><a href="https://uqam.ca/">Université du Québec<br>à Montréal</a></h6>
        </div>

        <div id="w-node-0d05fd79f6eb-1853825c" class="div-block">
          <div class="cards-image-mask"><img src="images/news_logo.png" srcset="images/news_logo-p-500.png 500w, images/news_logo-p-800.png 800w, images/news_logo.png 900w" sizes="(max-width: 479px) 55vw, (max-width: 767px) 57vw, (max-width: 991px) 137.59375px, 171.984375px" alt="" class="cards-image"></div>
          <h6 id="w-node-0d05fd79f6ee-1853825c" class="heading-6"><a href="https://paige.ai/" class="link-3">PAIGE</a></h6>
        </div>

        
         <! -- sp212 added -->
          <div id="w-node-0d05fd79f6e4-1853825c" class="div-block-2">
          <div class="cards-image-mask"><img src="images/Google_Logo.png" alt="" height = "10" width = "5" class="cards-image"></div>
          <h6 class="heading-7"><a href="https://google.com/">Google</a></h6>
          </div>
        
          <div id="w-node-0d05fd79f6e4-1853825c" class="div-block">
          <div class="portrait"><img src="images/AWS_healthcare_squid.png" alt=""  ></div>
          <h6 class="heading-7"><a href="https://aws.amazon.com/health/">AWS Healthcare</a></h6>
          </div>
         <! -- sp212 add end -->

        
        <div id="w-node-0d05fd79f6f2-1853825c" class="div-block-3">
          <div class="cards-image-mask"></div>
        </div>
      </div>
    </div>
  </section>
  <main id="main-content" class="main-content-2">
    <div class="w-container">
      <div class="rich-text-block-3 w-richtext">
        <h4>Previous workshops</h4>
        <ul>
          <li><a href="https://sites.google.com/view/icml-compbio-2019/">ICML 2019 Workshop on Computational Biology</a></li>
          <li><a href="https://sites.google.com/view/wcb2018/">Joint ICML and IJCAI 2018 Workshop on Computational Biology</a></li>
          <li><a href="https://www.google.com/url?q=https%3A%2F%2Fsites.google.com%2Fview%2Fcompbioworkshopicml2017%2Fhome&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNFWzRemcmyQ-NMnfr5nzoYSaBOgcA">ICML 2017 Workshop on Computational Biology</a></li>
          <li><a href="https://www.google.com/url?q=https%3A%2F%2Fsites.google.com%2Fsite%2Fcompbioworkshopicml2016%2F&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNFjnboz8_oyLofUYTQv1Aq_MxW0nA">ICML 2016 Workshop on Computational Biology</a></li>
        </ul>
      </div>
    </div>
  </main>
  <footer id="contact" class="footer">
    <div class="container-3 w-container">
      <h4 class="heading-3">Contact<a href="mailto:workshopcompbio@gmail.com"></a></h4>
    </div>
    <div class="w-container">
      <p class="paragraph-3">For workshop-related queries please contact: <a href="mailto:workshopcompbio@gmail.com?subject=Question%20about%20ICML%202020%20Computational%20Biology%20workshop%20">workshopcompbio@gmail.com</a></p>
    </div>
    <div class="w-container">
      <div class="text-block">Copyright © 2020 ICML compbio workshop. All rights reserved.<br>Designed by Yubin Xie</div>
    </div>
  </footer>
<!--   <script src="https://d3e54v103j8qbb.cloudfront.net/js/jquery-3.4.1.min.220afd743d.js?site=5e9514078c78742fa6aa5dc3" type="text/javascript" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script> -->
  <script src="js/webflow.js" type="text/javascript"></script>
  <!-- [if lte IE 9]><script src="https://cdnjs.cloudflare.com/ajax/libs/placeholders/3.0.2/placeholders.min.js"></script><![endif] -->
</body>
</html>
 <br>
